{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "fc5ef52b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: biopython in c:\\users\\gorke\\anaconda3\\lib\\site-packages (1.85)\n",
      "Requirement already satisfied: pandas in c:\\users\\gorke\\anaconda3\\lib\\site-packages (1.4.4)\n",
      "Requirement already satisfied: matplotlib in c:\\users\\gorke\\anaconda3\\lib\\site-packages (3.5.2)\n",
      "Requirement already satisfied: seaborn in c:\\users\\gorke\\anaconda3\\lib\\site-packages (0.11.2)\n",
      "Collecting scikit-bio\n",
      "  Using cached scikit-bio-0.6.3.tar.gz (5.4 MB)\n",
      "  Installing build dependencies: started\n",
      "  Installing build dependencies: finished with status 'done'\n",
      "  Getting requirements to build wheel: started\n",
      "  Getting requirements to build wheel: finished with status 'done'\n",
      "  Preparing metadata (pyproject.toml): started\n",
      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
      "Collecting matplotlib-venn\n",
      "  Using cached matplotlib-venn-1.1.2.tar.gz (40 kB)\n",
      "  Installing build dependencies: started\n",
      "  Installing build dependencies: finished with status 'done'\n",
      "  Getting requirements to build wheel: started\n",
      "  Getting requirements to build wheel: finished with status 'done'\n",
      "  Preparing metadata (pyproject.toml): started\n",
      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
      "Requirement already satisfied: numpy in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from biopython) (1.21.5)\n",
      "Requirement already satisfied: pytz>=2020.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from pandas) (2022.1)\n",
      "Requirement already satisfied: python-dateutil>=2.8.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from pandas) (2.8.2)\n",
      "Requirement already satisfied: packaging>=20.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (21.3)\n",
      "Requirement already satisfied: cycler>=0.10 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (0.11.0)\n",
      "Requirement already satisfied: pillow>=6.2.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (9.2.0)\n",
      "Requirement already satisfied: pyparsing>=2.2.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (3.0.9)\n",
      "Requirement already satisfied: fonttools>=4.22.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (4.25.0)\n",
      "Requirement already satisfied: kiwisolver>=1.0.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib) (1.4.2)\n",
      "Requirement already satisfied: scipy>=1.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from seaborn) (1.9.1)\n",
      "Requirement already satisfied: decorator>=3.4.2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (5.1.1)\n",
      "Collecting statsmodels>=0.14.0\n",
      "  Using cached statsmodels-0.14.5-cp39-cp39-win_amd64.whl (9.7 MB)\n",
      "Requirement already satisfied: h5py>=3.6.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (3.7.0)\n",
      "Requirement already satisfied: requests>=2.20.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (2.28.1)\n",
      "Collecting natsort>=4.0.3\n",
      "  Using cached natsort-8.4.0-py3-none-any.whl (38 kB)\n",
      "Collecting biom-format>=2.1.16\n",
      "  Using cached biom-format-2.1.16.tar.gz (11.7 MB)\n",
      "  Installing build dependencies: started\n",
      "  Installing build dependencies: finished with status 'done'\n",
      "  Getting requirements to build wheel: started\n",
      "  Getting requirements to build wheel: finished with status 'done'\n",
      "  Preparing metadata (pyproject.toml): started\n",
      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
      "Collecting pandas\n",
      "  Using cached pandas-2.3.1-cp39-cp39-win_amd64.whl (11.4 MB)\n",
      "Requirement already satisfied: patsy>=0.5.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (0.5.2)\n",
      "Collecting tzdata>=2022.7\n",
      "  Downloading tzdata-2025.2-py2.py3-none-any.whl (347 kB)\n",
      "     -------------------------------------- 347.8/347.8 kB 1.1 MB/s eta 0:00:00\n",
      "Collecting numpy\n",
      "  Using cached numpy-2.0.2-cp39-cp39-win_amd64.whl (15.9 MB)\n",
      "Requirement already satisfied: click in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from biom-format>=2.1.16->scikit-bio) (8.0.4)\n",
      "Requirement already satisfied: six in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from patsy>=0.5.0->scikit-bio) (1.16.0)\n",
      "Requirement already satisfied: idna<4,>=2.5 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (3.3)\n",
      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (1.26.11)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (2022.9.14)\n",
      "Requirement already satisfied: charset-normalizer<3,>=2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (2.0.4)\n",
      "  Downloading numpy-1.24.4-cp39-cp39-win_amd64.whl (14.9 MB)\n",
      "     --------------------------------------- 14.9/14.9 MB 12.8 MB/s eta 0:00:00\n",
      "Collecting patsy>=0.5.0\n",
      "  Downloading patsy-1.0.1-py2.py3-none-any.whl (232 kB)\n",
      "     ------------------------------------- 232.9/232.9 kB 13.9 MB/s eta 0:00:00\n",
      "Requirement already satisfied: colorama in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from click->biom-format>=2.1.16->scikit-bio) (0.4.5)\n",
      "Building wheels for collected packages: scikit-bio, matplotlib-venn, biom-format\n",
      "  Building wheel for scikit-bio (pyproject.toml): started\n",
      "  Building wheel for scikit-bio (pyproject.toml): finished with status 'error'\n",
      "  Building wheel for matplotlib-venn (pyproject.toml): started\n",
      "  Building wheel for matplotlib-venn (pyproject.toml): finished with status 'done'\n",
      "  Created wheel for matplotlib-venn: filename=matplotlib_venn-1.1.2-py3-none-any.whl size=45437 sha256=7bf2fce64f25bd6cd1d9a0835f48bf6bc155c5f13e1dd2d131c8f392efbda716\n",
      "  Stored in directory: c:\\users\\gorke\\appdata\\local\\pip\\cache\\wheels\\86\\29\\d8\\0f3c5a37c967a34fb40aaabd414f92104d2ad5fb149c0114a1\n",
      "  Building wheel for biom-format (pyproject.toml): started\n",
      "  Building wheel for biom-format (pyproject.toml): finished with status 'error'\n",
      "Successfully built matplotlib-venn\n",
      "Failed to build scikit-bio biom-format\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  error: subprocess-exited-with-error\n",
      "  \n",
      "  Building wheel for scikit-bio (pyproject.toml) did not run successfully.\n",
      "  exit code: 1\n",
      "  \n",
      "  [841 lines of output]\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:689:31: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:691:31: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:707:42: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:707:47: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:398:0: Exception check on '_dist_euclidean' will always require the GIL to be acquired. Declare '_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:407:0: Exception check on '_norm_euclidean' will always require the GIL to be acquired. Declare '_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:414:0: Exception check on '_sum' will always require the GIL to be acquired. Declare '_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:350:36: Exception check after calling '__pyx_fuse_0_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:357:24: Exception check after calling '__pyx_fuse_0_sum' will always require the GIL to be acquired. Declare '__pyx_fuse_0_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:381:35: Exception check after calling '__pyx_fuse_0_norm_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:389:34: Exception check after calling '__pyx_fuse_0_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:350:36: Exception check after calling '__pyx_fuse_1_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:357:24: Exception check after calling '__pyx_fuse_1_sum' will always require the GIL to be acquired. Declare '__pyx_fuse_1_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:381:35: Exception check after calling '__pyx_fuse_1_norm_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:389:34: Exception check after calling '__pyx_fuse_1_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:15: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:22: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:15: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:22: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:14: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:21: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:14: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:21: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:418:18: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:418:18: Use boundscheck(False) for faster access\n",
      "  [1/7] Cythonizing skbio/metadata/_intersection.pyx\n",
      "  [2/7] Cythonizing skbio/stats/ordination/_cutils.pyx\n",
      "  [3/7] Cythonizing skbio/tree/_c_nj.pyx\n",
      "  [4/7] Cythonizing skbio/alignment/_ssw_wrapper.pyx\n",
      "  [5/7] Cythonizing skbio/diversity/_phylogenetic.pyx\n",
      "  [6/7] Cythonizing skbio/stats/distance/_cutils.pyx\n",
      "  [7/7] Cythonizing skbio/tree/_c_me.pyx\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-gv4z6row\\overlay\\Lib\\site-packages\\setuptools\\_distutils\\dist.py:289: UserWarning: Unknown distribution option: 'tests_require'\n",
      "    warnings.warn(msg)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-gv4z6row\\overlay\\Lib\\site-packages\\setuptools\\dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please consider removing the following classifiers in favor of a SPDX license expression:\n",
      "  \n",
      "          License :: OSI Approved :: BSD License\n",
      "  \n",
      "          See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    self._finalize_license_expression()\n",
      "  running bdist_wheel\n",
      "  running build\n",
      "  running build_py\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\test.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\workflow.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_indexing.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_pairwise.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_path.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_tabular_msa.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_block.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_driver.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\_embedding.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\registry.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\util.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_fileobject.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_iosources.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\base.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\io.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\missing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_enan.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_interval.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\distance.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_alphabet.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_dna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_genetic_code.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_grammared_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_nucleotide_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_rna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_substitution.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\composition.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\gradient.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\power.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\_subsample.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  copying skbio\\table\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  copying skbio\\table\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\test_workflow.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_compare.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_majority_rule.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_me.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_nj.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_tree.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_upgma.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_decorator.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_docstring.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_plotting.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_pairwise.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_path.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_ssw.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_tabular_msa.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\\_lib\n",
      "  copying skbio\\alignment\\_lib\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\_lib\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_ace.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_chao1.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_gini.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_lladser.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_pd.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  copying skbio\\diversity\\beta\\_unifrac.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  copying skbio\\diversity\\beta\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_block.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_driver.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_ace.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_chao1.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_gini.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_lladser.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_pd.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  copying skbio\\diversity\\beta\\tests\\test_unifrac.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  copying skbio\\diversity\\beta\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\test_embedding.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\test_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\binary_dm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\biom.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\blast6.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\blast7.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\clustal.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\embed.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\embl.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\emptyfile.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\fasta.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\fastq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\genbank.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\gff3.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\lsmat.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\newick.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\ordination.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\phylip.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\qseq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\sample_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\stockholm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\taxdump.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_blast.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_sequence_feature_vocabulary.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_iosources.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_registry.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_binary_dm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_biom.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_blast6.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_blast7.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_clustal.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_embed.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_embl.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_emptyfile.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_fasta.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_fastq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_genbank.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_gff3.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_lsmat.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_newick.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_ordination.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_phylip.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_qseq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_sample_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_sequence_feature_vocabulary.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_stockholm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_taxdump.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_enan.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_intersection.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_interval.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_io.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_metadata_column.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_missing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_alphabet.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_distance.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_dna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_genetic_code.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_grammared_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_nucleotide_sequences.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_rna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_substitution.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_anosim.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_bioenv.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_mantel.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_permanova.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_permdisp.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  copying skbio\\stats\\evolve\\_hommola.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  copying skbio\\stats\\evolve\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_canonical_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_ordination_results.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_principal_coordinate_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_redundancy_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_composition.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_gradient.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_power.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_subsample.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_anosim.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_bioenv.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_mantel.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_permanova.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_permdisp.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  copying skbio\\stats\\evolve\\tests\\test_hommola.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  copying skbio\\stats\\evolve\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_canonical_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_ordination_results.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_principal_coordinate_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_redundancy_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  copying skbio\\table\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  copying skbio\\table\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_compare.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_majority_rule.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_me.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_nj.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_tree.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_upgma.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_decorator.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_docstring.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_plotting.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\faith-pd.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\otu-table.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\README.md -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\tree.nwk -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\otu-table.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\README.md -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\tree.nwk -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\unweighted_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\weighted_normalized_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\weighted_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\\data\n",
      "  copying skbio\\embedding\\tests\\data\\embed1.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file.bz2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file.gz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file.bz2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file.gz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\real_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\real_file_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_default_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_default_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_column_types -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_number_of_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_type_in_column -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_default_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_default_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_differing_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_for_sniffer -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_for_sniffer_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_gibberish -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_no_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_too_many_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_unrecognized_field -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embed1.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embed2.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_constructed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_feature_level_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_feature_level_record_no_FT -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_multi_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_lower -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_simple -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_upper -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_uniprot_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_w_beginning_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\empty -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_diff_ids.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_double_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_double_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_long_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_no_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_del.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_escape.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_null.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_space.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_tab.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_unit_sep.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_vtab.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_short_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_spaces.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_tabs.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_plus.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_plus.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_title.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_10_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_3_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_3_seqs_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_6_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_blanks_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_blank_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_after_10_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_line_within_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_legacy_format -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_first -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_last -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_middle -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_only_line_within_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_only_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_max_width_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_max_width_5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_mixed_qual_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq_roundtrip -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq_spaces -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_prot_seqs_odd_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_bio_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_bio_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_dna_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_dna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_prot_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_prot_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_rna_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_rna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_tabular_msa_different_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_blank_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_plus -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_in_seq_at_symbol -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_within_qual -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_missing_seq_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_plus -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_within_qual -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_seq_sanger -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_whitespace_stripping -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_single_seq_illumina1.3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_single_seq_illumina1.8 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_whitespace_only_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_as_illumina_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_as_sanger_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_original_sanger_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_illumina1.3_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_sanger_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_sanger_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_missing_locus_name -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_multi_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record_lower -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record_upper -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_w_beginning_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_bad_missing_directive -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_bad_wrong_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_dna -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_multi_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_original_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_and_blast7_default -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_invalid_differing_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_invalid_too_many_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error10 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error11 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error12 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error13 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error14 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error15 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error16 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error17 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error18 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error19 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error20 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error21 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error22 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error23 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error24 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error4 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error6 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error7 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error8 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error9 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_example2_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_example3_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_site_constraints -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_species -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_PCoA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_site_constraints -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_species -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_L&L_CA_data_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_PCoA_sample_data_3_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\pdb_hits.txt -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_dna_3_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_before_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_between_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_header_too_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_header_too_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_no_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_seq_too_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_seq_too_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_too_few_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_too_many_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_zero_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_zero_seq_len -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_single_seq_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_single_seq_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_two_chunks -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_variable_length_ids -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_varied_whitespace_in_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\prot.emb -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\prot_vec.emb -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_filter -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_lane -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_read -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_tile -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_x -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_y -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_multi_seq_illumina1.3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_single_seq_sanger -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_2_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_desc_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_extra -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_id_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_length_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_6_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_blanks_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_blank_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_line_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_legacy_format -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_first -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_last -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_middle -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_float -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_negative -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_over_255 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_string -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_whitespace_line_in_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_whitespace_only_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_ws_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_max_width_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_max_width_5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_multi_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_multi_seq_roundtrip -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_prot_seqs_odd_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_bio_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_dna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_nuc_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_prot_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_rna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_tabular_msa_different_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments-comment-char-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments-mixed-case.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-complete-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-leading-trailing-whitespace-split-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-leading-trailing-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\solexa_full_range_original_solexa.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_all_data_types -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_blank_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_data_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_different_padding -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_gc_data_length -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_gr_data_length -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_seq_lengths -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_gc -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_gr -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_sequence_names -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_tree_ids -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_extensive -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_extensive_mixed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_data_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_nonexistent_gr -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_nonexistent_gs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_data_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gc_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gf_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gr_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gs_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_metadata_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_footer -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_reference_items -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_rn_tag -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_msa -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_multi_line_trees -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_references -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_trees -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multi_line_tree_no_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multi_line_tree_with_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_nonstring_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_no_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_rna -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gf_no_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gf_with_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gs_no_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gs_with_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_references -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_references_mixed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_reference -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_tree_without_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_tree_with_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_two_of_each_metadata -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_whitespace_only_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\taxdump_names.dmp -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\taxdump_nodes.dmp -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tiny_embedding_file.npz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tsv_10_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tsv_8_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\whitespace_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\column-name-conflicts-with-id-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\comments-and-empty-rows-only.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\data-longer-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-after-directives-section.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-before-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-longer-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-column-names-with-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-column-names.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-directives.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-ids-with-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-ids.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-column-name.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-file -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\header-only-with-comments-and-empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\header-only.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\id-conflicts-with-id-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\invalid-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\missing-unknown-scheme.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\non-utf-8.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\qiime1-empty.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\simple-utf-16be.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\simple-utf-16le.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\types-directive-non-numeric.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\unrecognized-column-type.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\unrecognized-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\whitespace-only-column-name.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\whitespace-only-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\all-cells-padded.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\biom-observation-metadata.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\BOM-simple.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\case-insensitive-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\column-order.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\comments.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\complete-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\complete-types-directive_w_sk.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\empty-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\jagged-trailing-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\jagged.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\leading-trailing-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\mac-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\minimal.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-data.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-insdc-no-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-insdc.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-id-or-column-name-type-cast.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-newline-at-eof.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\non-standard-characters.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\numeric-column.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\override-insdc.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\partial-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiime1.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiita-preparation-information.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiita-sample-information.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\recommended-ids.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\rows-shorter-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple-with-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\single-column.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\single-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\trailing-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\unix-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\windows-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_data_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_data_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_no_data_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_no_data_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_wo_msg_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_wo_msg_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_w_msg_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_w_msg_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_real_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_real_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_df_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_exp_results_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\df.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\df_extra_column.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm2.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm3.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm4.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm_reordered.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results_different_column_order.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results_single_column.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\frameSeries_dm.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\frameSeries_grouping.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\mantel_env_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\mantel_veg_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\moving_pictures_dm.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\moving_pictures_mf.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_all_dms.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_dm_dm2.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_duplicate_dms.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_minimal.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_minimal_with_labels.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_na_p_value.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_reordered_distance_matrices.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_biplot_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_biplot_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_sample_constraints_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_sample_constraints_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_site_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_site_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_species_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_species_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_X -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_Y -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_biplot_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_biplot_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_sample_constraints_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_sample_constraints_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_site_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_site_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_species_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_species_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_X -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_Y -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\exp_PCoAEigenResults_site -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\exp_PCoAzeros_site -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\L&L_CA_data -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_biplot_descriptors -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_biplot_projected_descriptors -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_12dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3_eigh_ref_3dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3_fsvd_ref_3dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_skbio -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\varechem.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\varespec.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_biplot_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_constraints_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_eigvals_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_propexpl_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_sites_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_species_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  running build_ext\n",
      "  building 'skbio.metadata._intersection' extension\n",
      "  error: Microsoft Visual C++ 14.0 or greater is required. Get it with \"Microsoft C++ Build Tools\": https://visualstudio.microsoft.com/visual-cpp-build-tools/\n",
      "  [end of output]\n",
      "  \n",
      "  note: This error originates from a subprocess, and is likely not a problem with pip.\n",
      "  ERROR: Failed building wheel for scikit-bio\n",
      "  error: subprocess-exited-with-error\n",
      "  \n",
      "  Building wheel for biom-format (pyproject.toml) did not run successfully.\n",
      "  exit code: 1\n",
      "  \n",
      "  [345 lines of output]\n",
      "  <string>:15: SetuptoolsDeprecationWarning: The test command is disabled and references to it are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please remove any references to `setuptools.command.test` in all supported versions of the affected package.\n",
      "  \n",
      "          This deprecation is overdue, please update your project and remove deprecated\n",
      "          calls to avoid build errors in the future.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\setuptools\\_distutils\\dist.py:289: UserWarning: Unknown distribution option: 'tests_require'\n",
      "    warnings.warn(msg)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\setuptools\\dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please consider removing the following classifiers in favor of a SPDX license expression:\n",
      "  \n",
      "          License :: OSI Approved :: BSD License\n",
      "  \n",
      "          See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    self._finalize_license_expression()\n",
      "  running bdist_wheel\n",
      "  running build\n",
      "  running build_py\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\err.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\exception.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\parse.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\table.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\util.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\installation_informer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\metadata_adder.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\metadata_exporter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_converter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_head.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_ids.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_normalizer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_subsetter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_summarizer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_validator.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\uc_processor.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\util.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\long_lines.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_err.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_parse.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_add_metadata.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_show_install_info.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_subset_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_summarize_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_table_converter.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_table_normalizer.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_uc_processor.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_validate_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  running egg_info\n",
      "  writing biom_format.egg-info\\PKG-INFO\n",
      "  writing dependency_links to biom_format.egg-info\\dependency_links.txt\n",
      "  writing entry points to biom_format.egg-info\\entry_points.txt\n",
      "  writing requirements to biom_format.egg-info\\requires.txt\n",
      "  writing top-level names to biom_format.egg-info\\top_level.txt\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  reading manifest file 'biom_format.egg-info\\SOURCES.txt'\n",
      "  reading manifest template 'MANIFEST.in'\n",
      "  warning: no directories found matching 'support_files'\n",
      "  warning: no directories found matching 'tests\\test_data'\n",
      "  no previously-included directories found matching 'docs\\_build'\n",
      "  warning: no previously-included files matching '*.pyc' found anywhere in distribution\n",
      "  warning: no previously-included files matching '*.pyo' found anywhere in distribution\n",
      "  warning: no previously-included files matching '.git' found anywhere in distribution\n",
      "  warning: no previously-included files matching '*.so' found anywhere in distribution\n",
      "  warning: no previously-included files matching '.*.swp' found anywhere in distribution\n",
      "  adding license file 'COPYING.txt'\n",
      "  writing manifest file 'biom_format.egg-info\\SOURCES.txt'\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.assets' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.assets' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.assets' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.assets' to be distributed and are\n",
      "          already explicitly excluding 'biom.assets' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.tests.bench_tables' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.tests.bench_tables' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.tests.bench_tables' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.tests.bench_tables' to be distributed and are\n",
      "          already explicitly excluding 'biom.tests.bench_tables' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-4dc92cm_\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.tests.test_cli.test_data' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.tests.test_cli.test_data' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.tests.test_cli.test_data' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.tests.test_cli.test_data' to be distributed and are\n",
      "          already explicitly excluding 'biom.tests.test_cli.test_data' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  copying biom\\_filter.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\_subsample.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\_transform.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  copying biom\\assets\\exercise_api.py -> build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  copying biom\\assets\\exercise_cli.sh -> build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\json_obs_collapsed.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\json_sample_collapsed.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\test.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\test.json -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_data\\bad_table.txt -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\bad_table.txt.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\edgecase_issue_952.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\empty.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\no-contents.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.json -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.json.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test_grp_metadata.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  running build_ext\n",
      "  building 'biom._filter' extension\n",
      "  error: Microsoft Visual C++ 14.0 or greater is required. Get it with \"Microsoft C++ Build Tools\": https://visualstudio.microsoft.com/visual-cpp-build-tools/\n",
      "  [end of output]\n",
      "  \n",
      "  note: This error originates from a subprocess, and is likely not a problem with pip.\n",
      "  ERROR: Failed building wheel for biom-format\n",
      "ERROR: Could not build wheels for scikit-bio, biom-format, which is required to install pyproject.toml-based projects\n"
     ]
    }
   ],
   "source": [
    "!pip install  --user biopython pandas matplotlib seaborn scikit-bio matplotlib-venn\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "779527c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting biopython\n",
      "  Using cached biopython-1.85-cp39-cp39-win_amd64.whl (2.8 MB)\n",
      "Requirement already satisfied: numpy in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from biopython) (1.21.5)\n",
      "Installing collected packages: biopython\n",
      "Successfully installed biopython-1.85\n"
     ]
    }
   ],
   "source": [
    "!pip install biopython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "f32f35ab",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Biopython baaryla yklendi ve import edildi\n"
     ]
    }
   ],
   "source": [
    "\n",
    "from Bio import SeqIO\n",
    "print(\"Biopython baaryla yklendi ve import edildi\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "ae27aec8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting scikit-bio\n",
      "  Using cached scikit-bio-0.6.3.tar.gz (5.4 MB)\n",
      "  Installing build dependencies: started\n",
      "  Installing build dependencies: finished with status 'done'\n",
      "  Getting requirements to build wheel: started\n",
      "  Getting requirements to build wheel: finished with status 'done'\n",
      "  Preparing metadata (pyproject.toml): started\n",
      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
      "Collecting natsort>=4.0.3\n",
      "  Using cached natsort-8.4.0-py3-none-any.whl (38 kB)\n",
      "Requirement already satisfied: numpy>=1.17.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (1.21.5)\n",
      "Requirement already satisfied: decorator>=3.4.2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (5.1.1)\n",
      "Requirement already satisfied: scipy>=1.9.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (1.9.1)\n",
      "Requirement already satisfied: h5py>=3.6.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (3.7.0)\n",
      "Collecting statsmodels>=0.14.0\n",
      "  Using cached statsmodels-0.14.5-cp39-cp39-win_amd64.whl (9.7 MB)\n",
      "Collecting pandas>=1.5.0\n",
      "  Using cached pandas-2.3.1-cp39-cp39-win_amd64.whl (11.4 MB)\n",
      "Requirement already satisfied: patsy>=0.5.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (0.5.2)\n",
      "Requirement already satisfied: requests>=2.20.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from scikit-bio) (2.28.1)\n",
      "Collecting biom-format>=2.1.16\n",
      "  Using cached biom-format-2.1.16.tar.gz (11.7 MB)\n",
      "  Installing build dependencies: started\n",
      "  Installing build dependencies: finished with status 'done'\n",
      "  Getting requirements to build wheel: started\n",
      "  Getting requirements to build wheel: finished with status 'done'\n",
      "  Preparing metadata (pyproject.toml): started\n",
      "  Preparing metadata (pyproject.toml): finished with status 'done'\n",
      "Requirement already satisfied: click in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from biom-format>=2.1.16->scikit-bio) (8.0.4)\n",
      "Collecting numpy>=1.17.0\n",
      "  Using cached numpy-2.0.2-cp39-cp39-win_amd64.whl (15.9 MB)\n",
      "Requirement already satisfied: pytz>=2020.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from pandas>=1.5.0->scikit-bio) (2022.1)\n",
      "Requirement already satisfied: python-dateutil>=2.8.2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from pandas>=1.5.0->scikit-bio) (2.8.2)\n",
      "Collecting tzdata>=2022.7\n",
      "  Using cached tzdata-2025.2-py2.py3-none-any.whl (347 kB)\n",
      "Requirement already satisfied: six in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from patsy>=0.5.0->scikit-bio) (1.16.0)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (2022.9.14)\n",
      "Requirement already satisfied: idna<4,>=2.5 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (3.3)\n",
      "Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (1.26.11)\n",
      "Requirement already satisfied: charset-normalizer<3,>=2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from requests>=2.20.0->scikit-bio) (2.0.4)\n",
      "Collecting numpy>=1.17.0\n",
      "  Using cached numpy-1.24.4-cp39-cp39-win_amd64.whl (14.9 MB)\n",
      "Collecting patsy>=0.5.0\n",
      "  Using cached patsy-1.0.1-py2.py3-none-any.whl (232 kB)\n",
      "Requirement already satisfied: packaging>=21.3 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from statsmodels>=0.14.0->scikit-bio) (21.3)\n",
      "Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from packaging>=21.3->statsmodels>=0.14.0->scikit-bio) (3.0.9)\n",
      "Requirement already satisfied: colorama in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from click->biom-format>=2.1.16->scikit-bio) (0.4.5)\n",
      "Building wheels for collected packages: scikit-bio, biom-format\n",
      "  Building wheel for scikit-bio (pyproject.toml): started\n",
      "  Building wheel for scikit-bio (pyproject.toml): finished with status 'error'\n",
      "  Building wheel for biom-format (pyproject.toml): started\n",
      "  Building wheel for biom-format (pyproject.toml): finished with status 'error'\n",
      "Failed to build scikit-bio biom-format\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  error: subprocess-exited-with-error\n",
      "  \n",
      "  Building wheel for scikit-bio (pyproject.toml) did not run successfully.\n",
      "  exit code: 1\n",
      "  \n",
      "  [841 lines of output]\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:689:31: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:691:31: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:707:42: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  warning: skbio\\alignment\\_ssw_wrapper.pyx:707:47: Item lookup of unicode character codes now always converts to a Unicode string. Use an explicit C integer cast to get back the previous integer lookup behaviour.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:398:0: Exception check on '_dist_euclidean' will always require the GIL to be acquired. Declare '_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:407:0: Exception check on '_norm_euclidean' will always require the GIL to be acquired. Declare '_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:414:0: Exception check on '_sum' will always require the GIL to be acquired. Declare '_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:350:36: Exception check after calling '__pyx_fuse_0_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:357:24: Exception check after calling '__pyx_fuse_0_sum' will always require the GIL to be acquired. Declare '__pyx_fuse_0_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:381:35: Exception check after calling '__pyx_fuse_0_norm_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:389:34: Exception check after calling '__pyx_fuse_0_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_0_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:350:36: Exception check after calling '__pyx_fuse_1_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:357:24: Exception check after calling '__pyx_fuse_1_sum' will always require the GIL to be acquired. Declare '__pyx_fuse_1_sum' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:381:35: Exception check after calling '__pyx_fuse_1_norm_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_norm_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:389:34: Exception check after calling '__pyx_fuse_1_dist_euclidean' will always require the GIL to be acquired. Declare '__pyx_fuse_1_dist_euclidean' as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:15: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:22: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:15: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:403:22: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:14: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:21: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:14: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:411:21: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:418:18: Use boundscheck(False) for faster access\n",
      "  performance hint: skbio\\stats\\distance\\_cutils.pyx:418:18: Use boundscheck(False) for faster access\n",
      "  [1/7] Cythonizing skbio/metadata/_intersection.pyx\n",
      "  [2/7] Cythonizing skbio/stats/ordination/_cutils.pyx\n",
      "  [3/7] Cythonizing skbio/tree/_c_nj.pyx\n",
      "  [4/7] Cythonizing skbio/alignment/_ssw_wrapper.pyx\n",
      "  [5/7] Cythonizing skbio/diversity/_phylogenetic.pyx\n",
      "  [6/7] Cythonizing skbio/stats/distance/_cutils.pyx\n",
      "  [7/7] Cythonizing skbio/tree/_c_me.pyx\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-tz5mm_yy\\overlay\\Lib\\site-packages\\setuptools\\_distutils\\dist.py:289: UserWarning: Unknown distribution option: 'tests_require'\n",
      "    warnings.warn(msg)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-tz5mm_yy\\overlay\\Lib\\site-packages\\setuptools\\dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please consider removing the following classifiers in favor of a SPDX license expression:\n",
      "  \n",
      "          License :: OSI Approved :: BSD License\n",
      "  \n",
      "          See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    self._finalize_license_expression()\n",
      "  running bdist_wheel\n",
      "  running build\n",
      "  running build_py\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\test.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\workflow.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  copying skbio\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_indexing.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_pairwise.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_path.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\_tabular_msa.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  copying skbio\\alignment\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_block.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_driver.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  copying skbio\\diversity\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\_embedding.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  copying skbio\\embedding\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\registry.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\util.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_fileobject.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_iosources.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  copying skbio\\io\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\base.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\io.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\missing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_enan.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_interval.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  copying skbio\\metadata\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\distance.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_alphabet.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_dna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_genetic_code.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_grammared_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_nucleotide_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_repr.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_rna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\_substitution.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  copying skbio\\sequence\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\composition.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\gradient.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\power.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\_subsample.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  copying skbio\\stats\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  copying skbio\\table\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  copying skbio\\table\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\test_workflow.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  copying skbio\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_compare.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_majority_rule.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_me.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_nj.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_tree.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_upgma.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  copying skbio\\tree\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_decorator.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_docstring.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_exception.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_plotting.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  copying skbio\\util\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_pairwise.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_path.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_ssw.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\test_tabular_msa.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  copying skbio\\alignment\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\alignment\\_lib\n",
      "  copying skbio\\alignment\\_lib\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\alignment\\_lib\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_ace.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_chao1.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_gini.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_lladser.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\_pd.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  copying skbio\\diversity\\alpha\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  copying skbio\\diversity\\beta\\_unifrac.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  copying skbio\\diversity\\beta\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_block.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_driver.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  copying skbio\\diversity\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_ace.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_chao1.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_gini.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_lladser.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\test_pd.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  copying skbio\\diversity\\alpha\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  copying skbio\\diversity\\beta\\tests\\test_unifrac.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  copying skbio\\diversity\\beta\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\test_embedding.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\test_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  copying skbio\\embedding\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\binary_dm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\biom.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\blast6.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\blast7.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\clustal.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\embed.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\embl.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\emptyfile.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\fasta.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\fastq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\genbank.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\gff3.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\lsmat.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\newick.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\ordination.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\phylip.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\qseq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\sample_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\stockholm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\taxdump.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_blast.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\_sequence_feature_vocabulary.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  copying skbio\\io\\format\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_iosources.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_registry.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  copying skbio\\io\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_binary_dm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_biom.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_blast6.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_blast7.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_clustal.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_embed.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_embl.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_emptyfile.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_fasta.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_fastq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_genbank.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_gff3.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_lsmat.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_newick.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_ordination.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_phylip.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_qseq.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_sample_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_sequence_feature_vocabulary.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_stockholm.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\test_taxdump.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  copying skbio\\io\\format\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_enan.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_intersection.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_interval.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_io.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_metadata.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_metadata_column.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_missing.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\test_mixin.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  copying skbio\\metadata\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_alphabet.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_distance.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_dna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_genetic_code.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_grammared_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_nucleotide_sequences.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_protein.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_rna.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_sequence.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\test_substitution.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  copying skbio\\sequence\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\sequence\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_anosim.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_bioenv.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_mantel.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_permanova.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_permdisp.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  copying skbio\\stats\\distance\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  copying skbio\\stats\\evolve\\_hommola.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  copying skbio\\stats\\evolve\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_canonical_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_ordination_results.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_principal_coordinate_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_redundancy_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\_utils.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  copying skbio\\stats\\ordination\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_composition.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_gradient.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_power.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\test_subsample.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  copying skbio\\stats\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_anosim.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_bioenv.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_mantel.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_permanova.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\test_permdisp.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  copying skbio\\stats\\distance\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  copying skbio\\stats\\evolve\\tests\\test_hommola.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  copying skbio\\stats\\evolve\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\evolve\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_canonical_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_correspondence_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_ordination_results.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_principal_coordinate_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_redundancy_analysis.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  copying skbio\\stats\\ordination\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  copying skbio\\table\\tests\\test_base.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  copying skbio\\table\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\table\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_compare.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_majority_rule.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_me.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_nj.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_tree.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\test_upgma.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  copying skbio\\tree\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\tree\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_decorator.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_docstring.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_misc.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_plotting.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_testing.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\test_warning.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  copying skbio\\util\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\skbio\\util\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\faith-pd.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\otu-table.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\README.md -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\\tree.nwk -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\alpha\\tests\\data\\qiime-191-tt\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\otu-table.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\README.md -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\tree.nwk -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\unweighted_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\weighted_normalized_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  copying skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\\weighted_unifrac_dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\diversity\\beta\\tests\\data\\qiime-191-tt\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\\data\n",
      "  copying skbio\\embedding\\tests\\data\\embed1.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\embedding\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file.bz2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\big5_file.gz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file.bz2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\example_file.gz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\real_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  copying skbio\\io\\tests\\data\\real_file_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_custom_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_default_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_default_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_column_types -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_number_of_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast6_invalid_type_in_column -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_custom_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_default_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_default_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_differing_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_for_sniffer -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_for_sniffer_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_gibberish -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_no_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_too_many_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\blast7_invalid_unrecognized_field -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embed1.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embed2.txt.npy -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_constructed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_feature_level_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_feature_level_record_no_FT -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_multi_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_lower -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_simple -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_single_record_upper -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_uniprot_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\embl_w_beginning_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\empty -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_diff_ids.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_double_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_double_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_long_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_no_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_del.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_escape.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_null.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_space.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_tab.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_unit_sep.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_qual_vtab.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_short_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_spaces.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_tabs.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_plus.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_at_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_plus.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_qual.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_seq.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\error_trunc_in_title.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_10_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_3_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_3_seqs_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_6_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_blanks_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_blank_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_description_newline_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_id_whitespace_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_after_10_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_line_within_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_blank_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_legacy_format -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_first -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_last -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_missing_seq_data_middle -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_only_line_within_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_invalid_whitespace_only_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_max_width_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_max_width_5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_mixed_qual_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq_roundtrip -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_multi_seq_spaces -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_prot_seqs_odd_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_bio_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_bio_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_dna_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_dna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_prot_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_prot_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_rna_seq_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_rna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_single_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_tabular_msa_different_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fasta_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_blank_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_plus -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_after_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_in_seq_at_symbol -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_within_qual -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_blank_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_missing_seq_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_plus -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_after_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_within_qual -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_invalid_ws_line_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_blank_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_seq_sanger -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_whitespace_stripping -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_multi_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_single_seq_illumina1.3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_single_seq_illumina1.8 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_whitespace_only_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_as_illumina_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_as_sanger_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_wrapping_original_sanger_no_description -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_illumina1.3_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_sanger_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\fastq_writer_sanger_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_missing_locus_name -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_multi_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record_lower -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_single_record_upper -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\genbank_w_beginning_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_bad_missing_directive -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_bad_wrong_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_dna -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_multi_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\gff3_single_record -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\illumina_full_range_original_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_and_blast7_default -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_invalid_differing_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_invalid_too_many_columns -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_mixed_nans -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_multi_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\legacy9_single_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\longreads_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_dna_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\misc_rna_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error10 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error11 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error12 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error13 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error14 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error15 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error16 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error17 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error18 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error19 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error20 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error21 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error22 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error23 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error24 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error4 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error6 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error7 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error8 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_error9 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_example2_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_example3_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_site_constraints -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_CCA_species -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_PCoA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_site -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_site_constraints -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_exp_Ordination_RDA_species -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_L&L_CA_data_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\ordination_PCoA_sample_data_3_scores -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\pdb_hits.txt -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_dna_3_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_before_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_empty_line_between_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_header_too_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_header_too_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_no_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_seq_too_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_seq_too_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_too_few_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_too_many_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_zero_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_invalid_zero_seq_len -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_single_seq_long -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_single_seq_short -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_two_chunks -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_variable_length_ids -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_varied_whitespace_in_seqs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_2 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\phylip_whitespace_in_header_3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\prot.emb -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\prot_vec.emb -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_filter -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_lane -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_read -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_tile -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_x -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_invalid_y -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_multi_seq_illumina1.3 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qseq_single_seq_sanger -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_2_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_desc_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_extra -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_id_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_defaults_length_mismatch -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_3_seqs_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_5_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_5_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_6_blanks_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_6_ws_lines_start_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_blanks_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_blank_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_description_newline_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_empty_str -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_multi_char -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_id_whitespace_replacement_none -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_line_within_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_blank_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_legacy_format -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_first -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_last -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_missing_qual_scores_middle -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_float -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_negative -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_over_255 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_qual_scores_string -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_whitespace_line_in_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_whitespace_only_sequence -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_invalid_ws_line_after_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_max_width_1 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_max_width_5 -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_multi_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_multi_seq_roundtrip -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_prot_seqs_odd_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_bio_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_dna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_nuc_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_prot_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_rna_seq_non_defaults -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_single_seq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_tabular_msa_different_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_ws_lines_between_records -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\qual_ws_lines_end_of_file -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments-comment-char-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments-mixed-case.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-comments.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-complete-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-leading-trailing-whitespace-split-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sample-metadata-leading-trailing-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\sanger_full_range_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\solexa_full_range_original_solexa.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_all_data_types -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_blank_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_data_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_different_padding -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_gc_data_length -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_gr_data_length -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_differing_seq_lengths -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_gc -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_gr -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_sequence_names -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_duplicate_tree_ids -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_extensive -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_extensive_mixed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_data_type -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_nonexistent_gr -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_invalid_nonexistent_gs -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_data_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gc_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gf_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gr_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_malformed_gs_line -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_metadata_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_minimal -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_footer -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_header -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_reference_items -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_missing_rn_tag -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_msa -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_multi_line_trees -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_references -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multiple_trees -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multi_line_tree_no_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_multi_line_tree_with_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_nonstring_labels -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_no_data -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_rna -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gf_no_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gf_with_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gs_no_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_gs_with_whitespace -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_references -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_runon_references_mixed -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_reference -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_tree_without_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_single_tree_with_id -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_two_of_each_metadata -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\stockholm_whitespace_only_lines -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\taxdump_names.dmp -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\taxdump_nodes.dmp -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tiny_embedding_file.npz -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tsv_10_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\tsv_8_fields -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\whitespace_only -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_as_illumina.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_as_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  copying skbio\\io\\format\\tests\\data\\wrapping_original_sanger.fastq -> build\\lib.win-amd64-cpython-39\\skbio\\io\\format\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\column-name-conflicts-with-id-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\comments-and-empty-rows-only.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\data-longer-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-after-directives-section.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-before-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\directive-longer-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-column-names-with-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-column-names.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-directives.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-ids-with-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\duplicate-ids.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-column-name.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-file -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\empty-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\header-only-with-comments-and-empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\header-only.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\id-conflicts-with-id-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\invalid-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\missing-unknown-scheme.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\non-utf-8.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\qiime1-empty.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\simple-utf-16be.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\simple-utf-16le.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\types-directive-non-numeric.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\unrecognized-column-type.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\unrecognized-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\whitespace-only-column-name.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  copying skbio\\metadata\\tests\\data\\invalid\\whitespace-only-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\invalid\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\all-cells-padded.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\biom-observation-metadata.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\BOM-simple.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\case-insensitive-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\column-order.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\comments.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\complete-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\complete-types-directive_w_sk.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\empty-rows.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\empty-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\jagged-trailing-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\jagged.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\leading-trailing-whitespace.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\mac-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\minimal.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-data.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-insdc-no-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\missing-insdc.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-id-or-column-name-type-cast.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\no-newline-at-eof.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\non-standard-characters.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\numeric-column.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\override-insdc.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\partial-types-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiime1.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiita-preparation-information.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\qiita-sample-information.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\recommended-ids.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\rows-shorter-than-header.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple-with-directive.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\simple.txt -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\single-column.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\single-id.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\trailing-columns.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\unix-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  copying skbio\\metadata\\tests\\data\\valid\\windows-line-endings.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\metadata\\tests\\data\\valid\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_data_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_data_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_no_data_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\cr_no_data_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_wo_msg_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_wo_msg_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_w_msg_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\gr_w_msg_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_real_out -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  copying skbio\\stats\\tests\\data\\vr_real_raw -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_df_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\bioenv_exp_results_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\df.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\df_extra_column.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm2.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm3.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm4.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\dm_reordered.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results_different_column_order.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\exp_results_single_column.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\frameSeries_dm.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\frameSeries_grouping.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\mantel_env_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\mantel_veg_dm_vegan.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\moving_pictures_dm.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\moving_pictures_mf.tsv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_all_dms.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_dm_dm2.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_duplicate_dms.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_minimal.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_minimal_with_labels.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_na_p_value.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  copying skbio\\stats\\distance\\tests\\data\\pwmantel_exp_results_reordered_distance_matrices.txt -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\distance\\tests\\data\n",
      "  creating build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_biplot_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_biplot_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_sample_constraints_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_sample_constraints_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_site_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_site_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_species_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_species_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_X -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example2_Y -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_biplot_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_biplot_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_sample_constraints_scaling1 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_sample_constraints_scaling2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_site_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_site_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_species_scaling1_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_species_scaling2_from_vegan -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_X -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\example3_Y -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\exp_PCoAEigenResults_site -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\exp_PCoAzeros_site -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\L&L_CA_data -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_biplot_descriptors -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_biplot_projected_descriptors -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_12dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_2 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3 -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3_eigh_ref_3dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_sample_data_3_fsvd_ref_3dim -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\PCoA_skbio -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\varechem.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\varespec.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_biplot_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_constraints_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_eigvals_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_propexpl_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_sites_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  copying skbio\\stats\\ordination\\tests\\data\\vare_rda_species_from_vegan.csv -> build\\lib.win-amd64-cpython-39\\skbio\\stats\\ordination\\tests\\data\n",
      "  running build_ext\n",
      "  building 'skbio.metadata._intersection' extension\n",
      "  error: Microsoft Visual C++ 14.0 or greater is required. Get it with \"Microsoft C++ Build Tools\": https://visualstudio.microsoft.com/visual-cpp-build-tools/\n",
      "  [end of output]\n",
      "  \n",
      "  note: This error originates from a subprocess, and is likely not a problem with pip.\n",
      "  ERROR: Failed building wheel for scikit-bio\n",
      "  error: subprocess-exited-with-error\n",
      "  \n",
      "  Building wheel for biom-format (pyproject.toml) did not run successfully.\n",
      "  exit code: 1\n",
      "  \n",
      "  [345 lines of output]\n",
      "  <string>:15: SetuptoolsDeprecationWarning: The test command is disabled and references to it are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please remove any references to `setuptools.command.test` in all supported versions of the affected package.\n",
      "  \n",
      "          This deprecation is overdue, please update your project and remove deprecated\n",
      "          calls to avoid build errors in the future.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\setuptools\\_distutils\\dist.py:289: UserWarning: Unknown distribution option: 'tests_require'\n",
      "    warnings.warn(msg)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\setuptools\\dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          Please consider removing the following classifiers in favor of a SPDX license expression:\n",
      "  \n",
      "          License :: OSI Approved :: BSD License\n",
      "  \n",
      "          See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    self._finalize_license_expression()\n",
      "  running bdist_wheel\n",
      "  running build\n",
      "  running build_py\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\err.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\exception.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\parse.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\table.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\util.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\installation_informer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\metadata_adder.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\metadata_exporter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_converter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_head.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_ids.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_normalizer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_subsetter.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_summarizer.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\table_validator.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\uc_processor.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\util.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  copying biom\\cli\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\cli\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\long_lines.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_err.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_parse.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\test_util.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  copying biom\\tests\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_add_metadata.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_show_install_info.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_subset_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_summarize_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_table_converter.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_table_normalizer.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_uc_processor.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\test_validate_table.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  copying biom\\tests\\test_cli\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\__init__.py -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  running egg_info\n",
      "  writing biom_format.egg-info\\PKG-INFO\n",
      "  writing dependency_links to biom_format.egg-info\\dependency_links.txt\n",
      "  writing entry points to biom_format.egg-info\\entry_points.txt\n",
      "  writing requirements to biom_format.egg-info\\requires.txt\n",
      "  writing top-level names to biom_format.egg-info\\top_level.txt\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\arrayscalars.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarrayobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ndarraytypes.h won't be automatically included in the manifest: the path must be relative\n",
      "  dependency C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\numpy\\_core\\include\\numpy\\ufuncobject.h won't be automatically included in the manifest: the path must be relative\n",
      "  reading manifest file 'biom_format.egg-info\\SOURCES.txt'\n",
      "  reading manifest template 'MANIFEST.in'\n",
      "  warning: no directories found matching 'support_files'\n",
      "  warning: no directories found matching 'tests\\test_data'\n",
      "  no previously-included directories found matching 'docs\\_build'\n",
      "  warning: no previously-included files matching '*.pyc' found anywhere in distribution\n",
      "  warning: no previously-included files matching '*.pyo' found anywhere in distribution\n",
      "  warning: no previously-included files matching '.git' found anywhere in distribution\n",
      "  warning: no previously-included files matching '*.so' found anywhere in distribution\n",
      "  warning: no previously-included files matching '.*.swp' found anywhere in distribution\n",
      "  adding license file 'COPYING.txt'\n",
      "  writing manifest file 'biom_format.egg-info\\SOURCES.txt'\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.assets' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.assets' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.assets' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.assets' to be distributed and are\n",
      "          already explicitly excluding 'biom.assets' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.tests.bench_tables' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.tests.bench_tables' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.tests.bench_tables' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.tests.bench_tables' to be distributed and are\n",
      "          already explicitly excluding 'biom.tests.bench_tables' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  C:\\Users\\gorke\\AppData\\Local\\Temp\\pip-build-env-lfe1uaxu\\overlay\\Lib\\site-packages\\setuptools\\command\\build_py.py:212: _Warning: Package 'biom.tests.test_cli.test_data' is absent from the `packages` configuration.\n",
      "  !!\n",
      "  \n",
      "          ********************************************************************************\n",
      "          ############################\n",
      "          # Package would be ignored #\n",
      "          ############################\n",
      "          Python recognizes 'biom.tests.test_cli.test_data' as an importable package[^1],\n",
      "          but it is absent from setuptools' `packages` configuration.\n",
      "  \n",
      "          This leads to an ambiguous overall configuration. If you want to distribute this\n",
      "          package, please make sure that 'biom.tests.test_cli.test_data' is explicitly added\n",
      "          to the `packages` configuration field.\n",
      "  \n",
      "          Alternatively, you can also rely on setuptools' discovery methods\n",
      "          (for example by using `find_namespace_packages(...)`/`find_namespace:`\n",
      "          instead of `find_packages(...)`/`find:`).\n",
      "  \n",
      "          You can read more about \"package discovery\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html\n",
      "  \n",
      "          If you don't want 'biom.tests.test_cli.test_data' to be distributed and are\n",
      "          already explicitly excluding 'biom.tests.test_cli.test_data' via\n",
      "          `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,\n",
      "          you can try to use `exclude_package_data`, or `include-package-data=False` in\n",
      "          combination with a more fine grained `package-data` configuration.\n",
      "  \n",
      "          You can read more about \"package data files\" on setuptools documentation page:\n",
      "  \n",
      "          - https://setuptools.pypa.io/en/latest/userguide/datafiles.html\n",
      "  \n",
      "  \n",
      "          [^1]: For Python, any directory (with suitable naming) can be imported,\n",
      "                even if it does not contain any `.py` files.\n",
      "                On the other hand, currently there is no concept of package data\n",
      "                directory, all directories are treated like packages.\n",
      "          ********************************************************************************\n",
      "  \n",
      "  !!\n",
      "    check.warn(importable)\n",
      "  copying biom\\_filter.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\_subsample.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  copying biom\\_transform.pyx -> build\\lib.win-amd64-cpython-39\\biom\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  copying biom\\assets\\exercise_api.py -> build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  copying biom\\assets\\exercise_cli.sh -> build\\lib.win-amd64-cpython-39\\biom\\assets\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10000x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\1000x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\100x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x1000x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x100x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.001_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.001_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.010_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.010_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.100_bench.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  copying biom\\tests\\bench_tables\\10x10x0.100_bench_hdf5.biom.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\bench_tables\n",
      "  creating build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\json_obs_collapsed.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\json_sample_collapsed.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\test.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_cli\\test_data\\test.json -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_cli\\test_data\n",
      "  copying biom\\tests\\test_data\\bad_table.txt -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\bad_table.txt.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\edgecase_issue_952.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\empty.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\no-contents.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.json -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test.json.gz -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  copying biom\\tests\\test_data\\test_grp_metadata.biom -> build\\lib.win-amd64-cpython-39\\biom\\tests\\test_data\n",
      "  running build_ext\n",
      "  building 'biom._filter' extension\n",
      "  error: Microsoft Visual C++ 14.0 or greater is required. Get it with \"Microsoft C++ Build Tools\": https://visualstudio.microsoft.com/visual-cpp-build-tools/\n",
      "  [end of output]\n",
      "  \n",
      "  note: This error originates from a subprocess, and is likely not a problem with pip.\n",
      "  ERROR: Failed building wheel for biom-format\n",
      "ERROR: Could not build wheels for scikit-bio, biom-format, which is required to install pyproject.toml-based projects\n"
     ]
    }
   ],
   "source": [
    "!pip install scikit-bio"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "cc02c10d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting matplotlib-venn\n",
      "  Using cached matplotlib_venn-1.1.2-py3-none-any.whl\n",
      "Requirement already satisfied: matplotlib in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib-venn) (3.5.2)\n",
      "Requirement already satisfied: numpy in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib-venn) (1.21.5)\n",
      "Requirement already satisfied: scipy in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib-venn) (1.9.1)\n",
      "Requirement already satisfied: python-dateutil>=2.7 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (2.8.2)\n",
      "Requirement already satisfied: pyparsing>=2.2.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (3.0.9)\n",
      "Requirement already satisfied: kiwisolver>=1.0.1 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (1.4.2)\n",
      "Requirement already satisfied: cycler>=0.10 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (0.11.0)\n",
      "Requirement already satisfied: fonttools>=4.22.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (4.25.0)\n",
      "Requirement already satisfied: pillow>=6.2.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (9.2.0)\n",
      "Requirement already satisfied: packaging>=20.0 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from matplotlib->matplotlib-venn) (21.3)\n",
      "Requirement already satisfied: six>=1.5 in c:\\users\\gorke\\anaconda3\\lib\\site-packages (from python-dateutil>=2.7->matplotlib->matplotlib-venn) (1.16.0)\n",
      "Installing collected packages: matplotlib-venn\n",
      "Successfully installed matplotlib-venn-1.1.2\n"
     ]
    }
   ],
   "source": [
    "!pip install matplotlib-venn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "446ccbb3",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Orijinal DNA: ATGCGTACGTTA\n",
      "RNA Sekans: AUGCGUACGUUA\n",
      "Protein Sekans: MRTL\n"
     ]
    }
   ],
   "source": [
    "from Bio.Seq import Seq\n",
    "\n",
    "# DNA dizisini tanmla\n",
    "dna_seq = Seq(\"ATGCGTACGTTA\")\n",
    "print(\"Orijinal DNA:\", dna_seq)\n",
    "\n",
    "# Transkripsiyon (DNA  RNA)\n",
    "rna_seq = dna_seq.transcribe()\n",
    "print(\"RNA Sekans:\", rna_seq)\n",
    "\n",
    "# Translasyon (RNA  Protein)\n",
    "protein_seq = rna_seq.translate()\n",
    "print(\"Protein Sekans:\", protein_seq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "9881c71a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "      rnek1  rnek2  rnek3\n",
      "Gen                         \n",
      "GenA     5.2     4.8     5.2\n",
      "GenB     3.4     5.2     2.2\n",
      "GenC     7.1     8.2     1.5\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAewAAAF2CAYAAACs16tFAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8qNh9FAAAACXBIWXMAAA9hAAAPYQGoP6dpAAA/TElEQVR4nO3de3wM1/8/8NfmtrmvJJKIhCQSQqLud22pooSi2mqp1r1xpykq2opLCYq2X6EubYO631I+WqpIleLToIKSECpKhBBJSGIl2fP7w0/6WQl2N8lOJvN6Ph7zeHTPzjnz3p3Ge8+ZM2dUQggBIiIiqtAspA6AiIiIno0Jm4iISAaYsImIiGSACZuIiEgGmLCJiIhkgAmbiIhIBpiwiYiIZIAJm4iISAaYsEmxdDodCgoKpA6DiMggTNikWDNmzIC1tTVu3boldShERM+k4tKkpFSpqalITU1Fo0aNYGVlJXU4RERPxR42mcWpU6cwZMgQBAQEwM7ODnZ2dqhduzbCwsJw7NgxSWJavnw5mjdvjszMTL3yTz75BDVr1oSVlRWqVKlSJscaOHAg/Pz8yqStkqhUKkybNs2g/UaPHl3ie1u2bIFKpcKvv/5atsH9j9zcXEybNq1cj0FUWbFbQeVu2bJlGD16NIKCgjBu3DiEhIRApVLh3LlzWL9+PZo3b47k5GQEBARIHSq2b9+OWbNm4eOPP0bXrl2hVqulDskgR44cgY+Pj9RhPFNubi6mT58OAGjfvr20wRDJDBM2lavff/8dI0eORLdu3bBlyxbY2NgUvdehQweMGjUKmzdvhp2dnYRR/uvMmTMAgLFjx8LDw0PiaAzXqlUrqUMgonLGIXEqV7Nnz4alpSWWLVuml6z/15tvvonq1avrlR07dgw9evSAq6srbG1t0bhxY2zatElvn5UrV0KlUiEuLg4jRoxA1apV4ebmht69eyM1NdXoWP38/PDJJ58AADw9PfWGmTdu3IjOnTvDy8sLdnZ2qFevHiZPnoycnJxi7axcuRJBQUFQq9WoV68eVq9eXeLxHjx4gM8++wx169aFWq2Gu7s7Bg0ahPT0dL399u/fj/bt28PNzQ12dnaoWbMmXn/9deTm5hbtY+iQuCkMORfp6ekYOXIkgoOD4ejoCA8PD3To0AEHDx4s2ufy5ctwd3cHAEyfPh0qlQoqlQoDBw4EAEybNg0qlQqnTp3Cm2++CY1GA1dXV4SHh6OgoABJSUno0qULnJyc4Ofnh3nz5unFcP/+fXz44Ydo1KhRUd3WrVtj+/btxT7To0sDy5YtQ506daBWqxEcHIwNGzaU8bdHVHbYw6ZyU1hYiLi4ODRr1gxeXl4G14uLi0OXLl3QsmVLLF26FBqNBhs2bMBbb72F3Nzcon/gHxk6dCi6deuGdevW4Z9//sHEiRPRv39/7N+/36h4Y2NjsXjxYnz77bfYvXs3NBpN0TDzhQsXEBoaivHjx8PBwQGJiYmYO3cu/vjjD73jrFy5EoMGDULPnj2xYMECZGVlYdq0adBqtbCw+Pf3sU6nQ8+ePXHw4EFMmjQJbdq0QUpKCiIjI9G+fXscO3YMdnZ2uHz5Mrp164YXXngB3333HapUqYJr165h9+7dePDgAezt7Y36jAAghCjxdjadTleszNBzkZGRAQCIjIxEtWrVcO/ePcTGxqJ9+/bYt28f2rdvDy8vL+zevRtdunTBkCFDMHToUAAoSuKP9OnTB/3790dYWBh++eUXzJs3D/n5+di7dy9GjhyJCRMmYN26dfjoo48QGBiI3r17AwC0Wi0yMjIwYcIEeHt748GDB9i7dy969+6NmJgYvPfee3rH2bFjB+Li4jBjxgw4ODhgyZIl6Nu3L6ysrPDGG28Y/b0SlTtBVE7S0tIEAPH2228Xe6+goEDk5+cXbTqdrui9unXrisaNG4v8/Hy9Ot27dxdeXl6isLBQCCFETEyMACBGjhypt9+8efMEAHH9+vWnxhcZGSkAiPT09KeWPU6n04n8/Hxx4MABAUAkJCQIIYQoLCwU1atXF02aNNH7PJcvXxbW1tbC19e3qGz9+vUCgNi6date2/Hx8QKAWLJkiRBCiC1btggA4uTJk0/9LABEZGTkU/d5tN+ztri4uKL9DT0Xj3t0fl9++WXx2muvFZWnp6c/MdZH3/2CBQv0yhs1aiQAiG3bthWV5efnC3d3d9G7d+8nftZHMQwZMkQ0bty42PdgZ2cn0tLS9PavW7euCAwMfGKbRFLikDhJomnTprC2ti7aFixYAABITk5GYmIi3nnnHQBAQUFB0RYaGorr168jKSlJr60ePXrovW7QoAEAICUlpczivXTpEvr164dq1arB0tIS1tbWaNeuHQDg3LlzAICkpCSkpqaiX79+UKlURXV9fX3Rpk0bvfZ27tyJKlWq4NVXX9X7jI0aNUK1atWKZlE3atQINjY2eP/997Fq1SpcunSp1J+lT58+iI+PL7bNnTtXbz9jz8XSpUvRpEkT2NrawsrKCtbW1ti3b1/R92Oo7t27672uV68eVCoVunbtWlRmZWWFwMDAYud48+bNaNu2LRwdHYti+Pbbb0uM4eWXX4anp2fRa0tLS7z11ltITk7G1atXjYqZyByYsKncVK1aFXZ2diUmznXr1iE+Ph47duzQK79x4wYAYMKECXoJ3draGiNHjgSAYguduLm56b1+NLM7Ly+vTD7HvXv38MILL+C///0vPvvsM/z666+Ij4/Htm3b9I5z+/ZtAEC1atWKtfF42Y0bN5CZmQkbG5tinzMtLa3oMwYEBGDv3r3w8PDAqFGjEBAQgICAAHz11Vcmfx53d3c0a9as2FarVq1iMQKGnYuFCxdixIgRaNmyJbZu3YqjR48iPj4eXbp0Mfo8uLq66r22sbGBvb09bG1ti5Xfv3+/6PW2bdvQp08feHt7Y82aNThy5Aji4+MxePBgvf0eedp5enQuiSoSXsOmcmNpaYkOHTpgz549uH79ut517ODgYAAPJyL9r6pVqwIAIiIiiq5NPi4oKKh8An6C/fv3IzU1Fb/++mtRrxpAsfu3H/1wSEtLK9bG42WPJsjt3r27xGM6OTkV/fcLL7yAF154AYWFhTh27BgWLVqE8ePHw9PTE2+//bapH+uZjDkXa9asQfv27fH111/rvX/37t1yi+9xa9asgb+/PzZu3Kg3wqHVakvc/2nn6fEfgUQVARM2lauIiAjs2rULw4cPx5YtW2Btbf3U/YOCglC7dm0kJCRg9uzZZory6R794//4PdnLli3Tex0UFAQvLy+sX78e4eHhRfVSUlJw+PBhvZnw3bt3x4YNG1BYWIiWLVsaFIelpSVatmyJunXrYu3atThx4kS5JmxjzoVKpSr2/Zw6dQpHjhxBjRo1isrKevTj8RhsbGz0knVaWlqJs8QBYN++fbhx40bRsHhhYSE2btyIgIAAWdzTTsrDhE3lqm3btli8eDHGjBmDJk2a4P3330dISAgsLCxw/fp1bN26FQDg7OxcVGfZsmXo2rUrXnnlFQwcOBDe3t7IyMjAuXPncOLECWzevNmsn6FNmzZwcXHB8OHDERkZCWtra6xduxYJCQl6+1lYWGDmzJkYOnQoXnvtNQwbNgyZmZmYNm1aseHXt99+G2vXrkVoaCjGjRuHFi1awNraGlevXkVcXBx69uyJ1157DUuXLsX+/fvRrVs31KxZE/fv38d3330HAOjYsWO5f3ZDz0X37t0xc+ZMREZGol27dkhKSsKMGTPg7++vNyPdyckJvr6+2L59O15++WW4urqiatWqZbIKXPfu3bFt2zaMHDkSb7zxBv755x/MnDkTXl5euHDhQrH9q1atig4dOuDTTz8tmiWemJjIW7uo4pJ61hspw8mTJ8WgQYOEv7+/UKvVwtbWVgQGBor33ntP7Nu3r9j+CQkJok+fPsLDw0NYW1uLatWqiQ4dOoilS5cW7fNolnh8fLxe3bi4uGKznUtizCzxw4cPi9atWwt7e3vh7u4uhg4dKk6cOCEAiJiYGL19v/nmG1G7dm1hY2Mj6tSpI7777jsxYMAAvVniQjyc6Tx//nzRsGFDYWtrKxwdHUXdunVFWFiYuHDhghBCiCNHjojXXntN+Pr6CrVaLdzc3ES7du3Ejh079NqCEbPER40aVeJ7mzdvLvF7M+RcaLVaMWHCBOHt7S1sbW1FkyZNxA8//FDi5967d69o3LixUKvVAoAYMGCAEOLJ3/2AAQOEg4NDsXjbtWsnQkJC9MrmzJkj/Pz8hFqtFvXq1RMrVqwoarek72HJkiUiICBAWFtbi7p164q1a9c+7esjkhQf/kFEiqNSqTBq1ChER0dLHQqRwThLnIiISAaYsImIiGSAk86ISHF4JZDkiD1sIiIiGWDCJiIikgEmbCIiIhlgwiYiIpKBSjnpLP3sH1KHQGaUpXZ/9k5UaWi06VKHQGbkHtyiXNv/0dr0ZxN0y0969k5lqFImbCIiIkOorFXP3qmC4JA4ERGRDLCHTUREimVhJZ8eNhM2EREplspaPgPN8omUiIiojFlYqUzejFFQUIBPPvkE/v7+sLOzQ61atTBjxgzodDqD22APm4iIFMtck87mzp2LpUuXYtWqVQgJCcGxY8cwaNAgaDQajBs3zqA2mLCJiEixzHUN+8iRI+jZsye6desGAPDz88P69etx7Ngxg9vgkDgREZEJtFotsrOz9TatVlvivs8//zz27duH8+fPAwASEhJw6NAhhIaGGnw8JmwiIlIslbXK5C0qKgoajUZvi4qKKvE4H330Efr27Yu6devC2toajRs3xvjx49G3b1+DY+WQOBERKVZphsQjIiIQHh6uV6ZWq0vcd+PGjVizZg3WrVuHkJAQnDx5EuPHj0f16tUxYMAAg47HhE1ERIqlsjQ9YavV6icm6MdNnDgRkydPxttvvw0AeO6555CSkoKoqCgmbCIiomexKEXCNkZubi4sLPSvQltaWvK2LiIioork1VdfxaxZs1CzZk2EhITgzz//xMKFCzF48GCD22DCJiIixVJZmKeHvWjRInz66acYOXIkbt68ierVqyMsLAxTp041uA2VEEKUY4yS4OM1lYWP11QWPl5TWcr78ZqHmzU3uW6bY/FlGMmzsYdNRESKZa5r2GWBCZuIiBTLXEPiZYEJm4iIFEtOPWyudEZERCQD7GETEZFilWbhFHNjwiYiIsVSWchnoJkJm4iIFIuTzoiIiGRATpPOmLCJiEix5NTDls/gPRERkYKxh01ERIrFSWdEREQyIKchcSZsIiJSLE46IyIikgH2sImIiGRATtew5RMpERGRgrGHTUREisUhcSIiIhmQU8Ku0EPiJ0+elDoEIiKqxFQWKpM3c6twCTsrKwtLlixBkyZN0LRpU6nDISKiSkxlYWHyZm4VJmHv378f/fv3h5eXFxYtWoTQ0FAcO3ZM6rCIiKgSs7BUmbyZm6TXsK9evYqVK1fiu+++Q05ODvr06YP8/Hxs3boVwcHBUoZGRERUoUjWww4NDUVwcDDOnj2LRYsWITU1FYsWLZIqHCIiUiA5XcOWrIe9Z88ejB07FiNGjEDt2rWlCoOIiBSMC6cY4ODBg7h79y6aNWuGli1bIjo6Gunp6VKFQ0RECiSnHrZkCbt169ZYsWIFrl+/jrCwMGzYsAHe3t7Q6XT45ZdfcPfuXalCIyIihWDCNoK9vT0GDx6MQ4cO4fTp0/jwww8xZ84ceHh4oEePHlKHR0RElRhv6zJRUFAQ5s2bh6tXr2L9+vVSh0NERFRhVMilSS0tLdGrVy/06tVL6lCIiKgSk9PSpJIn7JycHMyZMwf79u3DzZs3odPpit5TqVS4ePGihNEREVFlJqdZ4pIn7KFDh+LAgQN499134eXlBZVKPr92zOXbDdsQszFWr8y1igY7YqJL3P/AkXjE/rwPyX9fwYP8fPjX8MHgt19Dy8YNzBEulbFNGzdg1aqV6NmzF94PG/7E/eLi9mPrls1ITU2Fvb09mjZthiFDh8HZ2dmM0ZKx+PctMRnlHMkT9q5du/Djjz+ibdu2UodSofnX8MaX0ycXvbZ4yq/Ck2eT0LxhfYS90weODvb4af9v+Gj2QiyfOw11avmZIVoqK+fPJ2H37l3w9/d/6n5//XUGCxfMx7Bh76NFy1a4ffsWFkcvwv999SU++XSqmaIlU/HvWzocEjeCi4sLXF1dpQ6jwrO0tISbSxWD9h03pL/e67D+fXDwjxP4Pf5P/kHLSF5eHj6fNw9jxo7Dxg1Pn4SZmJgIDw9P9OjZCwBQrVo1dOkaiq1bNpshUiot/n1LR05D4pJHOnPmTEydOhW5ublSh1KhXb2ehp6Dx+DNsA8QuSAa19JuGlxXp9MhN+8+nJ0cyzFCKmtfL1mM5i1aoHHjJs/ct169YNy6dQvx8X9ACIE7d+7g90OH0Lx5CzNESqXFv28yhOQ97AULFuDixYvw9PSEn58frK2t9d4/ceKERJFVHMG1A/DJuOGoUb0aMjKzsGrzdoyImIHvv4qCxtnpmfU3bN+F+/e16NCG/3jLxYEDvyI5+QK+/Mqw9fWDg4MxcdIkzJ0ThQcPHqCwsBAtW7XC8BEjyzlSKi3+fUuLQ+JGKO2tW1qtFlqtVr/swQOobWxK1W5F0rppw6L/DvCtgfpBgXhrxATsijuEt3t2fWrdXw4ewXcbtyEq4gO4VNGUd6hUBtLT07F82VLM/Gw2bAz8//jKlRQsW/o1+vbthyZNmyIjIwPfffsNoqP/D+PHh5dzxFQa/PuWlrmGxP38/JCSklKsfOTIkVi8eLFBbUiesCMjI0tVPyoqCtOnT9crmzByKCaNGlaqdisyO1tb1PL1wdXraU/db9+ho5gT/Q1mThyD5g3rmyk6Kq3kCxeQmZmJcWNHF5XpdDqcOXMG//nPDvyw/T+wtLTUq7Np40bUCw7G62+8CQDw968FW1tbTJo4Ae+9NwCurm5m/QxkOv59m5e5etjx8fEoLCwsen3mzBl06tQJb775psFtSJ6wASAzMxNbtmzBxYsXMXHiRLi6uuLEiRPw9PSEt7f3U+tGREQgPFy/B5F96VR5hiu5B/n5SLmaiob1gp64zy8HjyAqegWmhY9Em2aNzBcclVrDRo2weMlSvbIvv1gAH58aeOPNPsWSNfBwpOnx8kczjYUov1ip7PHv27zMlbDd3d31Xs+ZMwcBAQFo166dwW1InrBPnTqFjh07QqPR4PLlyxg2bBhcXV0RGxuLlJQUrF69+qn11Wo11Gq1Xpm2Eg2HA0D0ynVo26wxPN3dcCcrG6s2b0dObh66vvQCAGDp9xuRnnEHn457eI/uLweP4LOvlmHckP4IqROI23cyAQBqGxs4OthL9THIQPb29vDz89Mrs7W1hbOzc1H5ypjvcPv2bXw4YSIAoEXLllj0f1/hxx93okmTpriTkYHly5eiTp0guLmxd12R8e9bYqUYEi/pkmxJOelxDx48wJo1axAeHm7U2iOSJ+zw8HAMHDgQ8+bNg5PTvxMsunbtin79+kkYWcWRfjsD0xYuQdbdu6ji7IyQOgFYNncaqnlUBQDcvpOJG+m3i/bf/vN+FBYWYuHyVVi4fFVRedeXnsfHY8PMHj+VvYw7GUhP/3cmcadOnZGXl4ed/9mBb79ZAQcHBzRo2BCDBg2RMEoyBP++5aukS7KRkZGYNm3aU+v98MMPyMzMxMCBA406nkoIaQfMNBoNTpw4gYCAADg5OSEhIQG1atVCSkoKgoKCcP/+faPbTD/7RzlEShVVltr92TtRpaHRpksdApmRe3D5zn5P/2SQyXWdP11qUg/7lVdegY2NDf7zn/8YdTzJe9i2trbIzs4uVp6UlFRszJ+IiKgslWaWuCHJ+XEpKSnYu3cvtm3bZvTxJF84pWfPnpgxYwby8/MBPHzgx5UrVzB58mS8/vrrEkdHRESVmcpCZfJmipiYGHh4eKBbt25G15U8Yc+fPx/p6enw8PBAXl4e2rVrh8DAQDg6OmLWrFlSh0dERJWZhYXpm5F0Oh1iYmIwYMAAWFkZP8At+ZC4s7MzDh06hLi4OBw/fhw6nQ5NmjRBx44dpQ6NiIgqOXOudLZ3715cuXIFgwcPNqm+ZD3svLw87Ny5s+j1nj17kJqairS0NPz000+YNGmSSRPOiIiIKqLOnTtDCIE6deqYVF+yHvbq1auxc+dOdO/eHQAQHR2NkJAQ2NnZAXj49CEvLy988MEHUoVIRESVnEol+ZVhg0kW6dq1a4sNC6xbtw5xcXGIi4vD559/jk2bNkkUHRERKYKFyvTN3KGa/Yj/3/nz5/WGBWxtbfUe2t6iRQucPXtWitCIiEghVBYWJm/mJtmQeFZWlt4sufR0/cUQdDpdsRvSiYiIypKcHq8pWQ/bx8cHZ86ceeL7p06dgo+PjxkjIiIixVFZmL6ZmWQJOzQ0FFOnTi1xJnheXh6mT59u0o3lRERElZFkQ+JTpkzBpk2bEBQUhNGjR6NOnTpQqVRITExEdHQ0CgoKMGXKFKnCIyIiBZDTkLhkCdvT0xOHDx/GiBEjMHnyZDx6BolKpUKnTp2wZMkSeHp6ShUeEREpgQSTx0wl6Upn/v7+2L17NzIyMpCcnAwACAwMhKurq5RhERGRQhjzPGqpSb40KQC4urqiRYvyfYQaERFRMexhExERVXxyuoYtn58WRERECsYeNhERKZeM1hJnwiYiIuWS0ZA4EzYRESmWnJ7WxYRNRETKxR42ERFRxSfFU7dMJZ9IiYiIFIw9bCIiUi6udEZERCQDMhoSZ8ImIiLlYg+biIio4pPTpDMmbCIiUi4Z3Yctn0iJiIgUjD1sIiJSLi6cQkREVPFxaVIiIiI5YA+biIhIBmTUw5ZPpERERArGHjYRESkXF04hIiKSAS6cQkREJAMyuobNhE1ERMrFWeJEREQyIKMetnwiJSIiUjAmbCIiUi6VyvTNSNeuXUP//v3h5uYGe3t7NGrUCMePHze4PofEiYhIucw0S/zOnTto27YtXnrpJezatQseHh64ePEiqlSpYnAbTNhERKRcZroPe+7cuahRowZiYmKKyvz8/Ixqg0PiRESkXCoLkzetVovs7Gy9TavVlniYHTt2oFmzZnjzzTfh4eGBxo0bY8WKFUaFyoRNRETKZWFh8hYVFQWNRqO3RUVFlXiYS5cu4euvv0bt2rXx888/Y/jw4Rg7dixWr15tcKgqIYQoq89dUaSf/UPqEMiMstTuUodAZqTRpksdApmRe3CLcm3//u5vTK6reundYj1qtVoNtVpdbF8bGxs0a9YMhw8fLiobO3Ys4uPjceTIEYOOVymvYW+7Wr4nmCqW3j78gaYk47/3ljoEMqO1JXdYy04prmE/KTmXxMvLC8HBwXpl9erVw9atWw0+XqVM2ERERAYx08Ipbdu2RVJSkl7Z+fPn4evra3AbvIZNRETKZab7sD/44AMcPXoUs2fPRnJyMtatW4fly5dj1KhRBrfBhE1ERMpViklnxmjevDliY2Oxfv161K9fHzNnzsSXX36Jd955x+A2OCRORESKJcz4POzu3buje/fuJtdnD5uIiEgG2MMmIiLlktHTupiwiYhIuZiwiYiIKj5zXsMuLSZsIiJSLvawiYiIZEBGPWz5/LQgIiJSMPawiYhIuYxcAEVKTNhERKRYnHRGREQkB5x0RkREVPEJJmwiIiIZkNGQuHx+WhARESkYe9hERKRYHBInIiKSAxkNiTNhExGRcrGHTUREVPHxPmwiIiI5kFEPWz6REhERKRh72EREpFgCHBInIiKq8HhbFxERkRwwYRMREVV8nCVOREQkA3IaEpdPpERERArGHjYRESkXh8SJiIgqPjkNiTNhExGRYvE+bCIiIhlgD5uIiEgOeA3bcLdv34abmxsA4J9//sGKFSuQl5eHHj164IUXXpA4OiIioopBsrGA06dPw8/PDx4eHqhbty5OnjyJ5s2b44svvsDy5cvx0ksv4YcffpAqPCIiUgABC5M3c5MsYU+aNAnPPfccDhw4gPbt26N79+4IDQ1FVlYW7ty5g7CwMMyZM0eq8IiISAGESmXyZm6SDYnHx8dj//79aNCgARo1aoTly5dj5MiRsLB4+BtizJgxaNWqlVThERGRAnDSmQEyMjJQrVo1AICjoyMcHBzg6upa9L6Liwvu3r0rVXhERKQAcrqty6SfFjk5Ofj000/Rpk0bBAYGolatWnqboVSPDSk8/pqIiKg8CZWFyZsxpk2bBpVKpbc96rQayqQe9tChQ3HgwAG8++678PLyMjnRDhw4EGq1GgBw//59DB8+HA4ODgAArVZrUptEREQVUUhICPbu3Vv02tLS0qj6JiXsXbt24ccff0Tbtm1NqQ4AGDBggN7r/v37F9vnvffeM7l9IiKiZzHn5DErKyuje9V69U2p5OLione92RQxMTGlqk9ERFRapbmGrdVqi40Gq9XqopHjx124cAHVq1eHWq1Gy5YtMXv2bKMuI5t0DXvmzJmYOnUqcnNzTalORERUIZTmGnZUVBQ0Go3eFhUVVeJxWrZsidWrV+Pnn3/GihUrkJaWhjZt2uD27dsGx6oSQghjP2Djxo1x8eJFCCHg5+cHa2trvfdPnDhhcFs5OTmYM2cO9u3bh5s3b0Kn0+m9f+nSJWPDw7I9Rlep0BIOrkPCofXIzrgGAHCrVhutuoyEf0i7Z9a9duk4Nn31Lqp61ca7k7eXd6iS6O3zh9QhlKlvN2xDzMZYvTLXKhrsiIkucf8DR+IR+/M+JP99BQ/y8+FfwweD334NLRs3MEe4Zjf+e2+pQyhTPdo5oll9O1R3t8KDfIELKQ+wYXc2rt8qeGKdZiG26NjSAb5e1rC2UuHqzQJs3ZuN0xcq39yftVHle76vXDhncl3PmrWM6mH/r5ycHAQEBGDSpEkIDw836HgmDYn36tXLlGolKqsJbJWZY5VqeL7HBLi41wQA/PXfH7B9xSj0/ygWVb1qP7GeNu8udn//EWrWaY3cu7fMFS6VAf8a3vhy+uSi14/WJyjJybNJaN6wPsLe6QNHB3v8tP83fDR7IZbPnYY6tfzMEC2VRt1aauw9koOLVx/A0gLo84oGkwe7YdIXN6HNL7k/VddfjTPJWmzak42cPB3aNXXAhPfcMHVJOlKu55v5E8hbae7DNjQ5l8TBwQHPPfccLly4YHAdkxJ2ZGSkKdVKVBYT2Cq7gOc66L1+/tUPkHBoPa5fPvnUhL13w1TUbdodKgtLXDy194n7UcVjaWkJN5cqBu07boj+hM2w/n1w8I8T+D3+TyZsGZgXoz8kumzLHSz9xAv+3tZIvPygxDprdmbpvd60JxtNg23RpJ4tE7ZMaLVanDt3zqhnZpj80yIzMxPffPMNIiIikJGRAeDhUPi1a9eMaqcsJrApiU5XiMTjP6LgQS6q+zV+4n5njm5F5q0raN11tBmjo7Jy9Xoaeg4egzfDPkDkgmhcS7tpcF2dTofcvPtwdnIsxwipvNjbPhxlvJene8ae/1KpAFu1yqg69JCAyuTNGBMmTMCBAwfw999/47///S/eeOMNZGdnF7tj6mlM6mGfOnUKHTt2hEajweXLlzFs2DC4uroiNjYWKSkpWL16tcFtPZrAtmrVKtjb25sSjiKkpyZhw4K3UVCghY3aHq8OXQw3r8AS971z8zIO7ViAt8avhYWl5A9kIyMF1w7AJ+OGo0b1asjIzMKqzdsxImIGvv8qChpnp2fW37B9F+7f16JDmxZmiJbK2juhGiT+rcXVG0++hv240OcdobaxwH9P5ZVjZJWTuZYmvXr1Kvr27Ytbt27B3d0drVq1wtGjR+Hr62twGyb9ax4eHo6BAwdi3rx5cHL69x+Qrl27ol+/fka1tWDBAly8eBGenp4mTWAraVp9/gM1rG1Mu65QUbl6+KP/5B+gzcvGhZN78POaj9Bn7JpiSVunK8RPqz5E69AxcPHwlyhaKo3WTRsW/XeAbw3UDwrEWyMmYFfcIbzds+tT6/5y8Ai+27gNUREfwKWKprxDpTI2sIcGNb2sMWNpusF1Wje0Q++OTli4OgPZOexhG8tcS5Nu2LCh1G2YlLDj4+OxbNmyYuXe3t5IS0szqq3STmCLiorC9OnT9cq69Y/Eq+9OK1W7FY2llQ1c3B/+EqtW8zncSDmNEwdWo9PbM/T2e3A/BzeunMHNq+ewf/NMAIAQOkAIfDEuGK+P/BY1g1qbPX4ynZ2tLWr5+uDq9af/be07dBRzor/BzIlj0LxhfTNFR2XlvVc1aFLPFjOX30JGtmGJt9VzdhjWuwr+b10G/rpY+WaIm4MUT90ylUkJ29bWFtnZ2cXKk5KS4O7ublRbpZ3AFhERUWxK/OrfKlfvuiQCAoX5xSekqG0d8V7Ef/TKEg6uw5XzR/HqkP+Dxs3HXCFSGXmQn4+Uq6loWC/oifv8cvAIoqJXYFr4SLRp1sh8wVGZGNBDg2bBdvhsRTrS7xQaVKd1Qzu8/7oLojdk4GQSk7WphKjkCbtnz56YMWMGNm3aBODhQzuuXLmCyZMn4/XXXze6vczMTGzZsgUXL17ExIkT4erqihMnTsDT0xPe3k+/B6+kafXWNkaHUKEd2rEQfsEvwsmlGh5oc5B0/CdcvfAHeo/8BgBwcMcC3Mu8ga7vzYPKwgJVq9fRq2/n5AYra3WxcqqYoleuQ9tmjeHp7oY7WdlYtXk7cnLz0PWlh7NJl36/EekZd/DpuOEAHibrz75ahnFD+iOkTiBu38kEAKhtbODowHkhFd3Anhq0aWiPhd/fxn2tgMbx4TXV3Ps65P//y9hvveIMF2dLLN18B8DDZD38TRd8vzMLyVceFNV5kC+QpzV6aQ2SCZMS9vz58xEaGgoPDw/k5eWhXbt2SEtLQ6tWrTBr1iyj2irLCWyVVc7dW9j9/STkZN+Eja0T3KsHoffIb+Bb9+GtcDlZ6bh757rEUVJZSb+dgWkLlyDr7l1UcXZGSJ0ALJs7DdU8qgIAbt/JxI30f28F2v7zfhQWFmLh8lVYuHxVUXnXl57Hx2PDzB4/GadTq4ez+T99X390ctnmO/jtxMPVJKs4WcCtyr8PiujQwgFWlioM6lkFg3pWKSr/7XgOlm3JLPeYKxNh+s1SZmfSSmePxMXF4fjx49DpdGjSpAk6duxodBsdO3ZEkyZNiiawJSQkoFatWjh8+DD69euHy5cvG91mZVvpjJ6usq10Rk9X2VY6o6cr75XOzl+8YnLdOgE1yzCSZzPqp0VeXh527txZ9HrPnj1ITU1FWloafvrpJ0yaNAn37983KoD4+HiEhRXvBZgygY2IiMgY5roPuywYNSS+evVq7Ny5E927dwcAREdHIyQkBHZ2dgCAxMREeHl54YMPPjC4zbKcwEZERGQMKRKvqYzqYa9duxaDBw/WK1u3bh3i4uIQFxeHzz//vGgimqEeTWDLz3+4nF5pJ7AREREZSk49bKMS9vnz51Gnzr8zjW1tbfUeStCiRQucPXvWqADmz5+P9PR0vQlsgYGBcHR0NHoCGxERUWVl1JB4VlYWrKz+rZKerr8aj06nK7bq2LM4Ozvj0KFDZTKBjYiIyBhyug/bqB62j48Pzpw588T3T506BR8fwxbmKI8JbERERMaQ05C4UT3s0NBQTJ06Fd26dYOtra3ee3l5eZg+fTq6detmUFvlMYGNiIjIGHKadGZUwp4yZQo2bdqEoKAgjB49GnXq1IFKpUJiYiKio6NRUFCAKVOmGNTW2rVriyXjdevWoVatWgCANWvWYPHixUzYRERUbiptwvb09MThw4cxYsQITJ48GY/WXFGpVOjUqROWLFkCT09Pg9oyZALbqFGjjAmPiIio0jJ6aVJ/f3/s3r0bGRkZSE5OBgAEBgbC1dXVqHbKYwIbERGRMeQ06cyktcQBwNXVFS1atDD5wI8msAUFlfwEImMmsBEREZlCJ6MhcclWPX80ga2kmeDGTmAjIiIyRaWdJV6WynICGxERkSkUMSReWmU5gY2IiMgUlXaWeFkrqwlsRERElZ2kCfuR0k5gIyIiMgWHxImIiGSAQ+JEREQywB42ERGRDOikDsAITNhERKRYcuphS7ZwChERERmOPWwiIlIsTjojIiKSATkNiTNhExGRYrGHTUREJAM6IXUEhmPCJiIixZJTD5uzxImIiGSAPWwiIlIsTjojIiKSAcFr2ERERBWfjtewiYiIKj4hVCZvpoqKioJKpcL48eONqsceNhERKZa5h8Tj4+OxfPlyNGjQwOi67GETERGZwb179/DOO+9gxYoVcHFxMbo+EzYRESmWgMrkTavVIjs7W2/TarVPPNaoUaPQrVs3dOzY0aRYmbCJiEixdML0LSoqChqNRm+Liooq8TgbNmzA8ePHn/i+IXgNm4iIFKs0k8ciIiIQHh6uV6ZWq4vt988//2DcuHHYs2cPbG1tTT4eEzYRESlWaSadqdXqEhP0444fP46bN2+iadOmRWWFhYX47bffEB0dDa1WC0tLy2e2w4RNRESKZY77sF9++WWcPn1ar2zQoEGoW7cuPvroI4OSNcCETUREVK6cnJxQv359vTIHBwe4ubkVK38aJmwiIlIsLk1KREQkA1I9/OPXX381ug4TNhERKZaOPWwiIqKKj0PiREREMiD4tC4iIiIqS+xhExGRYvEaNhERkQzwGrbEev7SX+oQyIz++DJe6hDIjFK6LJc6BDIr73JtnQmbiIhIBnQS3YdtCiZsIiJSLDn1sDlLnIiISAbYwyYiIsWSUw+bCZuIiBSLt3URERHJgFQP/zAFEzYRESkWh8SJiIhkQE5D4pwlTkREJAPsYRMRkWJxSJyIiEgGmLCJiIhkQE7XsJmwiYhIsdjDJiIikgGdTuoIDMdZ4kRERDLAHjYRESkWh8SJiIhkgAmbiIhIBjhLnIiISAZEqbrY5n1wCBM2EREplpyGxDlLnIiISAbYwyYiIsWS033YTNhERKRYchoSZ8ImIiLF4ixxIiIiGWAPm4iISAZEqbrY5r2ti7PEiYiIZIA9bCIiUiw5XcNmD5uIiBRLCNM3Y3z99ddo0KABnJ2d4ezsjNatW2PXrl1GtcEeNhERKZbOTF1sHx8fzJkzB4GBgQCAVatWoWfPnvjzzz8REhJiUBtM2EREpFjmmiX+6quv6r2eNWsWvv76axw9epQJm4iI6FlKk7C1Wi20Wq1emVqthlqtfmq9wsJCbN68GTk5OWjdurXBx+M1bCIiIhNERUVBo9HobVFRUU/c//Tp03B0dIRarcbw4cMRGxuL4OBgg4/HHjYRESmWrhRd7IiICISHh+uVPa13HRQUhJMnTyIzMxNbt27FgAEDcODAAYOTtmQ97P379yM4OBjZ2dnF3svKykJISAgOHjwoQWRERKQUQmf6plari2Z9P9qelrBtbGwQGBiIZs2aISoqCg0bNsRXX31lcKySJewvv/wSw4YNg7Ozc7H3NBoNwsLCsHDhQgkiIyIipRBCmLyVxbEfvwb+NJIl7ISEBHTp0uWJ73fu3BnHjx83Y0RERKQ0Op3pmzGmTJmCgwcP4vLlyzh9+jQ+/vhj/Prrr3jnnXcMbkOya9g3btyAtbX1E9+3srJCenq6GSMiIiIqHzdu3MC7776L69evQ6PRoEGDBti9ezc6depkcBuSJWxvb2+cPn266Cbyx506dQpeXl5mjoqIiJSkLIa2DfHtt9+Wug3JhsRDQ0MxdepU3L9/v9h7eXl5iIyMRPfu3SWIjIiIlEInTN/MTbIe9ieffIJt27ahTp06GD16NIKCgqBSqXDu3DksXrwYhYWF+Pjjj6UKj4iIFKB0j9c0L8kStqenJw4fPowRI0YgIiKiaFhCpVLhlVdewZIlS+Dp6SlVeEREpADmWpq0LEi6cIqvry9++ukn3LlzB8nJyRBCoHbt2nBxcZEyLCIiUghzPfyjLFSIlc5cXFzQvHlzqcMgIiKqsCRP2Dk5OZgzZw727duHmzdvQvfYzW2XLl2SKLKKwz3iC1i6uhcrzzn8C+7GripWbuFUBU6v9oO1tz8sq3oi9/c9uLtjjTlCpTKgsrRE7alj4N33VairVYX2ejr+WR2L5NlLnjh+V61XJ9QM6wvnhvVgobbBvbMXcH5GNG79csjM0ZMpGoZo0K93DQQFOKKqmxoRs87g4NHbT9y/cX0NFkU1Klbeb8QfuHI1rxwjrXzMNUu8LEiesIcOHYoDBw7g3XffhZeXF1QqldQhVTi3/m8qVBb/Tui3quYD1/cjoE34o8T9VVZW0N27i3v7t8PhhScvTkMVU8DEYfB9/20kDP4Id88mQ9O0Php+E4WC7Lu4vGh1iXVcX2iOW3sPI+nTL5CfmY0aA3qj+Q9f4/e2fZB98pyZPwEZy87WEsl/38OPe9Mwe4phj1oEgL5hfyAnt6DodWZ2fnmEV6kJIxdAkZLkCXvXrl348ccf0bZtW6lDqbBEzl38729A9UuvouDWDTy4VPI/xIV3buHuju8BAPbN25khQipLVVo1wo3/7MPNXQcAAHkp11D9rW7QNK3/xDpnP5yt9zrp0y/g+erL8OjWgQlbBo4ez8DR4xlG17uT9QD3cgrLISLlKM3DP8xN8sdruri4wNXVVeow5MPSEnZN2iIv/oDUkVA5ufP7cbi91AoOtf0AAE4NguDatinSdxlxzlUqWDk5IP9OZrnESBXDd181xQ+rWuHLzxqg8XNVpA5HlqRcS9xYkvewZ86cialTp2LVqlWwt7eXOpwKzzakGVS29sg79pvUoVA5ufj5ClhpnNDuzC6IwkKoLC2R9OkXSN34o8Ft1PpgMCwd7HB9865yjJSkcuvOA8xdlISki/dgbWWBLh088NVnDTBmSgIS/sqSOjxZ4SxxIyxYsAAXL16Ep6cn/Pz8iq0vfuLEiafW12q1xZ52oi0ohNrKssxjrQjsWrSDNikBuuxMqUOhcuLVJxTe/Xrgz3c/xL2zyXBuWA/BCyJw//pNXPv+h2fWr/5WN9SeOhrHeo/Eg3Tjh1mp4vvnWh7+ufbv5LK/krLhUdUWfV/zYcKuxCRP2L169SpV/aioKEyfPl2v7MPWz2FC2walarcisqjiBpva9ZG5+kupQ6FyVG/OJFz8fDmub/oJAHD3zHnY1ayOwElhz0zYXm92RYPls3Di7XG4vf+IGaKliuKvpGx0bu8hdRiyI6NL2NIn7MjIyFLVj4iIQHh4uF7ZnciwUrVZUdk3bwfdvWxoz52UOhQqR5b2tsWWSxSFhYDF0++gqP5WNzRYMRt/9g8vmrBGylG7liNuZzyQOgzZ4dKkRsrMzMSWLVtw8eJFTJw4Ea6urjhx4gQ8PT3h7e391LpqtRpqtVqvLLcyDoerVLBr/iLyjh0s9iBWx659YKlxQdaGZUVlVtVrPqxmo4aFgxOsqteEKChA4c1Us4ZNxrvxYxwCJw/H/SupuHs2Gc6N6sF//CBcXbm1aJ+gz8Jh6+2JhEEfAXiYrBvGzMXZ8NnI/G8C1J5VAQCFefdRkH1Pks9BhrOztYC3l13Ray9PWwT6O+DuvQLcSNci7D1/uLvZ4LMvkgAAb/bwRtrN+/g7JRfW1ip0bu+Jl9q6Y8rsv6T6CLIlp1nikifsU6dOoWPHjtBoNLh8+TKGDRsGV1dXxMbGIiUlBatXl3zfqdLY1A6BpUvVEmeHWzpXgWWVqnplVT/49zYf6xq1YNekLQoz0pEe9UG5x0ql89e4zxA0fRxCFkVC7eGG+6k3cWXFRlz4bHHRPmovd9jV+PfxszWHvQULa2vUXxSJ+ov+HbX6Z/U2nBoSYdb4yXh1A530FkIZO/ThY4d/2peG2V8mwc3VBp7utkXvW1tZYNSgALi72UD7QIe/r+RiwrTTJt0apnRy6mGrhMTLvHTs2BFNmjTBvHnz4OTkhISEBNSqVQuHDx9Gv379cPnyZaPbTJvYv+wDpQrr+JfxUodAZhTVZbnUIZAZHfpP+a4lMWp+psl1F0+oUmZxGELy+7Dj4+MRFlb8mrO3tzfS0tIkiIiIiKjikXxI3NbWFtnZ2cXKk5KS4O5efP1sIiKisiKjEXHpe9g9e/bEjBkzkJ//cA1clUqFK1euYPLkyXj99dcljo6IiCozoRMmb+YmecKeP38+0tPT4eHhgby8PLRr1w6BgYFwdHTErFmzpA6PiIgqMS5NagRnZ2ccOnQIcXFxOH78OHQ6HZo0aYKOHTtKHRoREVVyclqaVLIedl5eHnbu3Fn0es+ePUhNTUVaWhp++uknTJo0Cffv35cqPCIiUgD2sA2wevVq7Ny5E927dwcAREdHIyQkBHZ2DxcPSExMhJeXFz74gPcNExERSdbDXrt2LQYPHqxXtm7dOsTFxSEuLg6ff/45Nm3aJFF0RESkBJx0ZoDz58+jTp06Ra9tbW1hYfFvOC1atMDZs2elCI2IiBRCTglbsiHxrKwsWFn9e/j09HS993U6XbHHZhIREZUlOa0lLlkP28fHB2fOnHni+6dOnYKPj48ZIyIiIqWRUw9bsoQdGhqKqVOnljgTPC8vD9OnT0e3bt0kiIyIiJSCs8QNMGXKFGzatAlBQUEYPXo06tSpA5VKhcTERERHR6OgoABTpkyRKjwiIqIKRbKE7enpicOHD2PEiBGYPHly0a8VlUqFTp06YcmSJfD09JQqPCIiUgA5LZwi6Upn/v7+2L17NzIyMpCcnAwACAwMhKurq5RhERGRQsjpediSL00KAK6urmjRooXUYRARkcJIcS3aVBUiYRMREUlB6HRSh2AwJmwiIlIsOV3DlvzxmkRERPRsTNhERKRY5roPOyoqCs2bN4eTkxM8PDzQq1cvJCUlGdUGEzYRESmWuVY6O3DgAEaNGoWjR4/il19+QUFBATp37oycnByD2+A1bCIiUixz3da1e/duvdcxMTHw8PDA8ePH8eKLLxrUBhM2EREplk6YPktcq9UWe0iVWq2GWq1+Zt2srCwAMGrdEQ6JExGRYpVmSDwqKgoajUZvi4qKevYxhUB4eDief/551K9f3+BY2cMmIiIyQUREBMLDw/XKDOldjx49GqdOncKhQ4eMOh4TNhERKVZprmEbOvz9v8aMGYMdO3bgt99+M/oR0kzYRESkWOZamlQIgTFjxiA2Nha//vor/P39jW6DCZuIiBRLZ6alSUeNGoV169Zh+/btcHJyQlpaGgBAo9HAzs7OoDaYsImISLHMdVvX119/DQBo3769XnlMTAwGDhxoUBtM2EREpFiiFLd1GXec0v8w4G1dREREMsAeNhERKZa5hsTLAhM2EREpFhM2ERGRDJRmaVJzY8ImIiLFYg+biIhIBoSZ7sMuC5wlTkREJAPsYRMRkWJxSJyIiEgGzLVwSllgwiYiIsXSsYdNRERU8clp0hkTNhERKZacrmFzljgREZEMsIdNRESKxUlnREREMiCnIXEmbCIiUiw5TTpTibJ4qjZJTqvVIioqChEREVCr1VKHQ+WM51tZeL4JYMKuNLKzs6HRaJCVlQVnZ2epw6FyxvOtLDzfBHCWOBERkSwwYRMREckAEzYREZEMMGFXEmq1GpGRkZyQohA838rC800AJ50RERHJAnvYREREMsCELZGPPvoIXbt2BQCEhIRgyZIlEkdE5YnnW1l4vqk8cEhcIrdu3YJWq4W3tzdSUlJQpUoVaDQaqcOicsLzrSw831QemLAV5vLly/D398eff/6JRo0aSR0OlTOeb2Xh+a7cOCQugX/++QdDhgxB9erVYWNjA19fX4wbNw63b9+WOjQAwPLly9G+fXs4OztDpVIhMzNT6pBkrSKf74yMDIwZMwZBQUGwt7dHzZo1MXbsWGRlZUkdmmxV5PMNAGFhYQgICICdnR3c3d3Rs2dPJCYmSh0WGYAJ28wuXbqEZs2a4fz581i/fj2Sk5OxdOlS7Nu3D61bt0ZGRkaJ9R48eGC2GHNzc9GlSxdMmTLFbMesrCr6+U5NTUVqairmz5+P06dPY+XKldi9ezeGDBliluNXNhX9fANA06ZNERMTg3PnzuHnn3+GEAKdO3dGYWGh2WIgEwkyqy5duggfHx+Rm5urV379+nVhb28vhg8fLoQQwtfXV8ycOVMMGDBAODs7i/fee0/ExMQIjUYjdu/eLerWrSscHBzEK6+8IlJTU/Xa+u6770TdunWFWq0WQUFBYvHixUXv/f333wKA+PPPP4UQQhQWFoqhQ4eK2rVri8uXL+u1ExcXJwCIO3fulP0XoRByOt+PbNq0SdjY2Ij8/Pwy/CaUQY7nOyEhQQAQycnJZfhNUHlgwjaj27dvC5VKJWbPnl3i+8OGDRMuLi5Cp9MJX19f4ezsLD7//HNx4cIFceHCBRETEyOsra1Fx44dRXx8vDh+/LioV6+e6NevX1Eby5cvF15eXmLr1q3i0qVLYuvWrcLV1VWsXLlSCKH/B63VasXrr78uGjVqJG7cuFEsHibs0pHb+X5kxYoVomrVqmX7ZSiAHM/3vXv3xPjx44W/v7/QarVl/6VQmWLCNqOjR48KACI2NrbE9xcuXCgAiBs3bghfX1/Rq1cvvfdjYmKK/RJevHix8PT0LHpdo0YNsW7dOr16M2fOFK1btxZC/PsHffDgQdGxY0fRtm1bkZmZWWI8TNilI7fzLYQQt27dEjVr1hQff/yxsR9X8eR0vhcvXiwcHBwEAFG3bl32rmXCyoyj7/QM4v9P2FepVACAZs2aFdvH3t4eAQEBRa+9vLxw8+ZNAEB6enrRhJdhw4YV7VNQUFDslpK+ffvCx8cH+/btg729fZl/Fnq2ina+s7Oz0a1bNwQHByMyMrJ0H46KqUjn+5133kGnTp1w/fp1zJ8/H3369MHvv/8OW1vb0n9QKjecdGZGgYGBUKlUOHv2bInvJyYmwsXFBVWrVgUAODg4FNvH2tpa77VKpSr6h0Cn0wEAVqxYgZMnTxZtZ86cwdGjR/XqhYaG4tSpU8XKqezI6XzfvXsXXbp0gaOjI2JjY4sdl55NTudbo9Ggdu3aePHFF7FlyxYkJiYiNjbWuA9MZseEbUZubm7o1KkTlixZgry8PL330tLSsHbtWrz11ltFv8CN5enpCW9vb1y6dAmBgYF6m7+/v96+I0aMwJw5c9CjRw8cOHDA5M9ETyaX852dnY3OnTvDxsYGO3bsYC/LRHI53yURQkCr1ZoUF5kPh8TNLDo6Gm3atMErr7yCzz77DP7+/vjrr78wceJEeHt7Y9asWaVqf9q0aRg7diycnZ3RtWtXaLVaHDt2DHfu3EF4eLjevmPGjEFhYSG6d++OXbt24fnnnwfw8B+XtLQ0JCcnAwBOnz4NJycn1KxZE66urqWKT2kq+vm+e/cuOnfujNzcXKxZswbZ2dnIzs4GALi7u8PS0rJU8SlNRT/fly5dwsaNG9G5c2e4u7vj2rVrmDt3Luzs7BAaGlqq2MgMJLx+rliXL18WAwcOFNWqVRPW1taiRo0aYsyYMeLWrVtF+/j6+oovvvhCr96j2z7+V2xsrHj8NK5du1Y0atRI2NjYCBcXF/Hiiy+Kbdu2CSGK3/YhhBALFiwQTk5O4vfffxdCCBEZGSkAFNtiYmLK7DtQkop8vh9NLCxp+/vvv8vya1CMiny+r127Jrp27So8PDyEtbW18PHxEf369ROJiYll+h1Q+eDSpERERDLAa9hEREQywIRNREQkA0zYREREMsCETUREJANM2ERERDLAhE1ERCQDTNhEREQywIRNREQkA0zYREREMsCETUREJANM2ERERDLAhE1ERCQD/w8imU8r87mk/AAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 600x400 with 2 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "\n",
    "data= {\n",
    "    'Gen' :['GenA','GenB','GenC'],\n",
    "    'rnek1':[5.2,3.4,7.1],\n",
    "    'rnek2':[4.8,5.2,8.2],\n",
    "    'rnek3':[5.2,2.2,1.5],\n",
    "}\n",
    "\n",
    "df= pd.DataFrame(data)\n",
    "\n",
    "df.set_index('Gen',inplace=True)\n",
    "print(df)\n",
    "\n",
    "plt.figure(figsize=(6,4))\n",
    "sns.heatmap(df,annot=True,cmap ='coolwarm')\n",
    "plt.title('Gen fadesi Heatmap')\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "eef99f73",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAqgAAAHWCAYAAAC7eHtwAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8qNh9FAAAACXBIWXMAAA9hAAAPYQGoP6dpAABOAElEQVR4nO3deVwW5f7/8ffNDiKgoIKKiOCCWy60uJRLbrjVscXMOloulXumlVkPrVA7maflmEtWUmnmt71jpqalZmqFe+4LKqa4oSAuNwLz+8Mf9+kWUIGb+x7k9Xw85iFzzXXPfEbOfXp7zcw1FsMwDAEAAAAm4ebqAgAAAIC/I6ACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAqgzMjJyVFWVparywAAXAcBFUCZ8corr8jT01OnTp1ydSkAgGuw8KpTAGXF0aNHdfToUTVp0kQeHh6uLgcAUABGUAGUiK1bt2rAgAGKioqSr6+vfH19Vbt2bT3xxBNKTEx0SU3vvfeebr31Vp09e9au/cUXX1SNGjXk4eGhoKAghxyrf//+qlmzpkP2lR+LxaKJEyfeUL/cxd3dXRUqVNAtt9yiJ554QuvXry+x+q6npP9+AJRuDCEAcLjZs2dr2LBhqlu3rkaOHKkGDRrIYrFo586dWrBggW699Vbt27dPUVFRri5V3377rSZNmqTx48crLi5O3t7eri7phqxbt07Vq1e/ob7333+/nnnmGRmGofT0dP3555/6+OOP9d5772nEiBF6++23S7haACgcAioAh/r11181ZMgQdevWTV988YW8vLxs29q3b6+hQ4fq888/l6+vrwur/J8///xTkjRixAhVrlzZxdXcuDvuuOOG+1apUsWuf+fOnTVq1CgNHjxY77zzjurVq6ennnqqJMoEgCLhEj8Ah5o8ebLc3d01e/Zsu3D6dw888ICqVq1q15aYmKiePXuqYsWK8vHxUdOmTfV///d/dn0SEhJksVj0888/66mnnlJISIiCg4PVq1cvHT16tNC11qxZUy+++KKkKyHu75fNFy5cqE6dOiksLEy+vr6KiYnR888/r/Pnz+fZT0JCgurWrStvb2/FxMTo448/zvd4mZmZio+PV7169eTt7a1KlSrpscce08mTJ+36/fTTT2rbtq2Cg4Pl6+urGjVq6L777tOFCxdsfW70En9B3N3dNX36dIWEhGjq1Km29kuXLumZZ55RkyZNFBgYqIoVK6pFixb69ttv8+zj7NmzGjBggCpWrCh/f39169ZNBw4cyFNb7u/t4MGD16ypZs2a6t+/v2195cqVslgs+vTTT/Xcc88pLCxM/v7+6tGjh44fP65z585p8ODBCgkJUUhIiB577DFlZGRcc58ASgdGUAE4THZ2tn7++WfFxsYqLCzshj/3888/q0uXLrr99ts1a9YsBQYG6rPPPlPv3r114cKFPAFj4MCB6tatmz799FMlJydr7NixeuSRR/TTTz8Vqt6vv/5a7777rj744AMtWbJEgYGBtsvme/fuVdeuXTVq1CiVK1dOu3bt0r/+9S/9/vvvdsdJSEjQY489pnvuuUfTpk1TWlqaJk6cKKvVKje3/40B5OTk6J577tEvv/yiZ599Vi1bttShQ4c0YcIEtW3bVomJifL19dXBgwfVrVs33Xnnnfrwww8VFBSkv/76S0uWLFFmZqb8/PwKdY7X4uvrqw4dOuizzz7TkSNHVL16dVmtVqWmpmrMmDGqVq2aMjMztXz5cvXq1Utz587VP//5T9v59OjRQ4mJiZo4caKaNWumdevWqUuXLg6rL9cLL7ygdu3aKSEhQQcPHtSYMWPUp08feXh46JZbbtGCBQu0adMmvfDCCypfvrzeeecdh9cAwMkMAHCQlJQUQ5Lx0EMP5dmWlZVlXL582bbk5OTYttWrV89o2rSpcfnyZbvPdO/e3QgLCzOys7MNwzCMuXPnGpKMIUOG2PV7/fXXDUnGsWPHrlnfhAkTDEnGyZMnr9l2tZycHOPy5cvGqlWrDEnGli1bDMMwjOzsbKNq1apGs2bN7M7n4MGDhqenpxEREWFrW7BggSHJ+PLLL+32/ccffxiSjBkzZhiGYRhffPGFIcnYvHnzNc9FkjFhwoRr9sntN3To0AK3P/fcc4Yk47fffst3e+7vbcCAAUbTpk1t7d9//70hyZg5c6Zd/ylTpuSpLff3lpSUZGvr16+f3d+PYRhGRESE0a9fP9v6zz//bEgyevToYddv1KhRhiRjxIgRdu333nuvUbFixWvuE0DpwCV+AE7RvHlzeXp62pZp06ZJkvbt26ddu3apb9++kqSsrCzb0rVrVx07dky7d++221fPnj3t1hs3bixJOnTokMPqPXDggB5++GGFhobK3d1dnp6eatOmjSRp586dkqTdu3fr6NGjevjhh2WxWGyfjYiIUMuWLe32t2jRIgUFBalHjx5259ikSROFhoZq5cqVkqQmTZrIy8tLgwcP1kcffaQDBw447JzyY+Qz0+Dnn3+uVq1ayd/fXx4eHvL09NQHH3xgO29JWrVqlSTpwQcftPtsnz59HF5j9+7d7dZjYmIkSd26dcvTnpqamucyP4DSh4AKwGFCQkLk6+ubb1D89NNP9ccff+i7776zaz9+/LgkacyYMXYB1tPTU0OGDJGkPBPrBwcH263nPnl/8eJFh5xHRkaG7rzzTv3222+Kj4/XypUr9ccff+irr76yO87p06clSaGhoXn2cXXb8ePHdfbsWXl5eeU5z5SUFNs5RkVFafny5apcubKGDh2qqKgoRUVFldiT9rm/q9x7gr/66is9+OCDqlatmubNm6d169bpjz/+0OOPP65Lly7ZPnf69Gl5eHioYsWKdvurUqWKw2u8+hi59zYX1P73OgGUTtyDCsBh3N3d1b59ey1btkzHjh2zuw+1fv36kpTnQZmQkBBJ0rhx49SrV69891u3bt2SKbgAP/30k44ePaqVK1faRk0l5Zk/NTcop6Sk5NnH1W25D3QtWbIk32OWL1/e9vOdd96pO++8U9nZ2UpMTNR//vMfjRo1SlWqVNFDDz1U1NPK4+LFi1q+fLmioqJs997OmzdPkZGRWrhwod2osNVqtftscHCwsrKylJqaahcU8/u7AIDCYgQVgEONGzdO2dnZevLJJ3X58uXr9q9bt65q166tLVu2KDY2Nt/l7+HNGXKD2dVzos6ePdtuvW7dugoLC9OCBQvsLpUfOnRIa9eutevbvXt3nT59WtnZ2fmeY34h3N3dXbfffrveffddSdLGjRsdcn7SlQfahg0bptOnT+u5556ztVssFnl5edmF05SUlDxP8ecG94ULF9q1f/bZZw6rEUDZxQgqAIdq1aqV3n33XQ0fPlzNmjXT4MGD1aBBA7m5uenYsWP68ssvJUkBAQG2z8yePVtxcXHq3Lmz+vfvr2rVqik1NVU7d+7Uxo0b9fnnnzv1HFq2bKkKFSroySef1IQJE+Tp6an58+dry5Ytdv3c3Nz06quvauDAgfrHP/6hQYMG6ezZs5o4cWKeS/wPPfSQ5s+fr65du2rkyJG67bbb5OnpqSNHjujnn3/WPffco3/84x+aNWuWfvrpJ3Xr1k01atTQpUuX9OGHH0qSOnToUKTzOX78uNavXy/DMHTu3DnbRP1btmzR008/rUGDBtn6du/eXV999ZWGDBmi+++/X8nJyXr11VcVFhamvXv32vp16dJFrVq10jPPPKP09HQ1b95c69ats02x9fcZDACgsAioABzuySefVIsWLfT222/rzTff1NGjR2WxWFS9enW1bNlSK1asUPv27W3927Vrp99//12TJk3SqFGjdObMGQUHB6t+/fp5HsJxhuDgYH3//fd65pln9Mgjj6hcuXK65557tHDhQjVr1syu74ABAyRJ//rXv9SrVy/VrFlTL7zwglatWmV78Em6Mhr63Xff6e2339Ynn3yiKVOmyMPDQ9WrV1ebNm3UqFEjSVceklq2bJkmTJiglJQU+fv7q2HDhvruu+/UqVOnIp3PF198oS+++EJubm7y9/dXRESEWrRooVmzZuWZ8P+xxx7TiRMnNGvWLH344YeqVauWnn/+eR05ckQvv/yyrZ+bm5v++9//6plnntFrr72mzMxMtWrVSvPmzdMdd9zhsFfGAiibLEZ+j3ACAFAEn376qfr27atff/01z0wGAHCjCKgAgCJZsGCB/vrrLzVq1Ehubm5av369pk6dqqZNm9qmoQKAouASPwCgSMqXL6/PPvtM8fHxOn/+vMLCwtS/f3/Fx8e7ujQApRwjqAAAADAVHrMEAACAqRBQAQAAYCoEVAAAAJjKTfGQVE5Ojo4ePary5cvbvf0EAAAA5pD7spCqVate92UeN0VAPXr0qMLDw11dBgAAAK4jOTlZ1atXv2afmyKg5r6nOzk52e71iQAAADCH9PR0hYeH23LbtdwUATX3sn5AQAABFQAAwMRu5HZMHpICAACAqRBQAQAAYCoEVAAAAJjKTXEP6o0wDENZWVnKzs52dSllmru7uzw8PJgODAAAFKhMBNTMzEwdO3ZMFy5ccHUpkOTn56ewsDB5eXm5uhQAAGBCN31AzcnJUVJSktzd3VW1alV5eXkxeucihmEoMzNTJ0+eVFJSkmrXrn3diXoBAEDZc9MH1MzMTOXk5Cg8PFx+fn6uLqfM8/X1laenpw4dOqTMzEz5+Pi4uiQAAGAyZWb4ipE68+B3AQAAroWk4CTPPfec4uLiJEkNGjTQjBkzXFwRAACAOd30l/jNYuzYsbJarZKkxYsXKygoyLUFAQAAmBQjqE4SEhKiatWqSZIiIiIUGBjolOMePHhQFotFmzdvdsrxAAAAiouA6gTJyckaMGCAbRaBiIgIjRw5UqdPn3Z1aQAAAKZDQC1hBw4cUGxsrPbs2aMFCxZo3759mjVrllasWKEWLVooNTU1389lZmY6uVIAAEqpnYuk99pJk8Ku/LlzkasrQjERUEvY0KFD5eXlpWXLlqlNmzaqUaOG4uLitHz5cv31118aP368JKlmzZqKj49X//79FRgYqEGDBikhIUFBQUFaunSpYmJi5O/vry5duujYsWN2x5g7d65iYmLk4+OjevXqXfMBrJycHA0aNEh16tTRoUOHSvTcAQAocTsXSQv7Skc3SpcvXPlz4SOE1FKOgFqCUlNTtXTpUg0ZMkS+vr5220JDQ9W3b18tXLhQhmFIkqZOnaqGDRtqw4YNeumllyRJFy5c0BtvvKFPPvlEq1ev1uHDhzVmzBjbfubMmaPx48dr0qRJ2rlzpyZPnqyXXnpJH330UZ56MjMz9eCDDyoxMVFr1qxRRERECZ49AABO8Mu0fBoNac2/nV4KHIen+EvQ3r17ZRiGYmJi8t0eExOjM2fO6OTJk5Kk9u3b24XPNWvW6PLly5o1a5aioqIkScOGDdMrr7xi6/Pqq69q2rRp6tWrlyQpMjJSO3bs0OzZs9WvXz9bv4yMDHXr1k0XL17UypUrnfaQFgAAJerkrvzbTxTQjlKBgOpCuSOnua9ejY2NzdPHz8/PFk4lKSwsTCdOnJAknTx50vYA1qBBg2x9srKy8gTQPn36qHr16lqxYgVv1AIA3Dwq1btyWf9qles5vxY4DJf4S1B0dLQsFot27NiR7/Zdu3apQoUKCgkJkSSVK1cuTx9PT0+7dYvFYgu2OTk5kq5c5t+8ebNt+fPPP7V+/Xq7z3Xt2lVbt27N0w4AQKl25zOSLFc1Wv5/O0orAmoJCg4OVseOHTVjxgxdvHjRbltKSormz5+v3r1720ZQC6tKlSqqVq2aDhw4oOjoaLslMjLSru9TTz2l1157TT179tSqVauKfE4AAJhKTHep9zypWnPJs9yVPx+aL9Xr5urKUAxc4i9h06dPV8uWLdW5c2fFx8crMjJS27dv19ixY1WtWjVNmjSpWPufOHGiRowYoYCAAMXFxclqtSoxMVFnzpzR6NGj7foOHz5c2dnZ6t69u3744Qe1bt26WMcGAMAUYrpfWXDTMMUIalZWll588UVFRkbK19dXtWrV0iuvvGK7hF2a1a5dW4mJiYqKilLv3r0VFRWlwYMHq127dlq3bp0qVqxYrP0PHDhQ77//vhISEtSoUSO1adNGCQkJeUZQc40aNUovv/yyunbtqrVr1xbr2AAAACXBYuTe0OhCkyZN0ptvvqmPPvpIDRo0UGJioh577DHFx8dr5MiR1/18enq6AgMDlZaWpoCAALttly5dUlJSkiIjI+Xj41NSp4BC4HcCAEDZc628djVTXOJft26d7rnnHnXrduV+kZo1a2rBggVKTEx0cWUAAABwNlNc4m/durVWrFihPXv2SJK2bNmiNWvWqGvXrvn2t1qtSk9Pt1sAAABwczDFCOpzzz2ntLQ01atXT+7u7srOztakSZPUp0+ffPtPmTJFL7/8spOrBAAAgDOYYgR14cKFmjdvnj799FNt3LhRH330kd544418X9cpSePGjVNaWpptSU5OdnLFAAAAKCmmGEEdO3asnn/+eT300EOSpEaNGunQoUOaMmWK3es6c3l7e8vb29vZZQIAAMAJTDGCeuHCBbm52Zfi7u5+U0wzBQAAgMIxxQhqjx49NGnSJNWoUUMNGjTQpk2b9O9//1uPP/64q0sDAACAk5liBPU///mP7r//fg0ZMkQxMTEaM2aMnnjiCb366quuLs0UnnvuOcXFxUmSGjRooBkzZri4IgAAgJJjion6i+tmn6j/1KlTslqtqlatmg4dOqSgoCAFBga6uqwiuxl+JwAAoHAKM1G/KUZQcW0hISGqVq2aJCkiIsJp4fTgwYOyWCzavHmzU44HAAAgEVBv2K6UdL21fI9e+e8OvbV8j3alOOflAMnJyRowYICqVq0qLy8vRUREaOTIkTp9+rRTjn897733ntq2bauAgABZLBadPXvW1SUBAIBSzhQPSZnZwVPnNfr/Nmvj4bNyd7PIzSLlGNJby/eqeUQFTXvgFtUMKVcixz5w4IBatGihOnXqaMGCBYqMjNT27ds1duxY/fDDD1q/fr0qVqyY53OZmZny8vIqkZquduHCBXXp0kVdunTRuHHjnHJMAABwc2ME9RoOnjqve979VVuOpEmSsnMMXc42lJ1z5bbdzclndc+7v+rgqfMlcvyhQ4fKy8tLy5YtU5s2bVSjRg3FxcVp+fLl+uuvvzR+/HhJUs2aNRUfH6/+/fsrMDBQgwYNUkJCgoKCgrR06VLFxMTI399fXbp00bFjx+yOMXfuXMXExMjHx0f16tW75gNYOTk5GjRokOrUqaNDhw5JkkaNGqXnn39ed9xxR4n8HQAAgLKHgHoNo/9vszKsWbZAerXsHEMZ1iyN+XyLw4+dmpqqpUuXasiQIfL19bXbFhoaqr59+2rhwoXKfcZt6tSpatiwoTZs2KCXXnpJ0pXRzTfeeEOffPKJVq9ercOHD2vMmDG2/cyZM0fjx4/XpEmTtHPnTk2ePFkvvfRSvm/wyszM1IMPPqjExEStWbNGERERDj9nAAAAiUv8BdqVkq6Nh89et192jqHEQ2e0KyVd9UKv/URaYezdu1eGYSgmJibf7TExMTpz5oxOnjwpSWrfvr1d+FyzZo0uX76sWbNmKSoqSpI0bNgwvfLKK7Y+r776qqZNm6ZevXpJkiIjI7Vjxw7Nnj3b7g1eGRkZ6tatmy5evKiVK1eW6hkEAACA+RFQC7DkzxS5u1kKHD39O3c3i5b+edyhAfV6ckdOLRaLJCk2NjZPHz8/P1s4laSwsDCdOHFCknTy5EnbA1iDBg2y9cnKysoTQPv06aPq1atrxYoV8vPzc/i5AAAA/B2X+AuQfjFLbpYb6+tmkdIuXnbo8aOjo2WxWLRjx458t+/atUsVKlRQSEiIJKlcubwPanl6etqtWywWW7DNfY3snDlztHnzZtvy559/av369Xaf69q1q7Zu3ZqnHQAAoCQQUAsQ4OuhGxg8lXTlqf5AX8/rdyyE4OBgdezYUTNmzNDFixfttqWkpGj+/Pnq3bu3bQS1sKpUqaJq1arpwIEDio6OtlsiIyPt+j711FN67bXX1LNnT61atarI5wQAAHAjuMRfgC4NQ/XW8r031Dc7x1CXhqEOr2H69Olq2bKlOnfurPj4eLtppqpVq6ZJkyYVa/8TJ07UiBEjFBAQoLi4OFmtViUmJurMmTMaPXq0Xd/hw4crOztb3bt31w8//KDWrVtLuhKWU1JStG/fPknStm3bVL58edWoUSPfKbAAAACuhxHUAtQLDVCzGkFyv851fnc3i2IjKqhuaHmH11C7dm0lJiYqKipKvXv3VlRUlAYPHqx27dpp3bp1xQ6AAwcO1Pvvv6+EhAQ1atRIbdq0UUJCQp4R1FyjRo3Syy+/rK5du2rt2rWSpFmzZqlp06a2+1jvuusuNW3aVN99912xagMAAGWXxci9KbEUu9a7XYvz3vfceVALmmrK3c0if28PfTu0VYlN1n8zKs7vBAAAlE7XymtXYwT1GmqGlNO3Q1upaXiQpCuB1NPdYhtVbRoeRDgFAABwMO5BvY6aIeX0xVMttSslXUv/PK60i5cV6OupLg1DS+SyPgAAQFlHQL1B9UIDnDrPKQAAQFnFJX4AAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBNRS4LnnnlNcXJwkqUGDBpoxY4aLKwIAACg5PMVfCowdO1ZWq1WStHjxYgUFBbm2IAAAgBJEQL1Rx7dLO/8rXUqTfAKlmB5SlQZOOXRISIjt54iICKccU5IOHjyoyMhIbdq0SU2aNHHacQEAQNnGJf7rOb1fer+jNLOltOp16fc5V/6c2VL6oNOV7SUoOTlZAwYMUNWqVeXl5aWIiAiNHDlSp0+fLtHj3ojU1FQNHz5cdevWlZ+fn2rUqKERI0YoLS3N1aUBAIBSjIB6Laf3S3Pulv7acGXdyJZyLl/5U5KOJF7ZXkIh9cCBA4qNjdWePXu0YMEC7du3T7NmzdKKFSvUokULpaam5vu5zMzMEqnnakePHtXRo0f1xhtvaNu2bUpISNCSJUs0YMAApxwfAADcnAio1/L1k5I1/X+B9GpG9pXt3zxVIocfOnSovLy8tGzZMrVp00Y1atRQXFycli9frr/++kvjx4+XJNWsWVPx8fHq37+/AgMDNWjQICUkJCgoKEhLly5VTEyM/P391aVLFx07dszuGHPnzlVMTIx8fHxUr169az6AlZOTo0GDBqlOnTo6dOiQGjZsqC+//FI9evRQVFSU2rdvr0mTJum///2vsrKySuTvBAAA3PwIqAU5vl068nvB4TSXkS0l/3alvwOlpqZq6dKlGjJkiHx9fe22hYaGqm/fvlq4cKEMw5AkTZ06VQ0bNtSGDRv00ksvSZIuXLigN954Q5988olWr16tw4cPa8yYMbb9zJkzR+PHj9ekSZO0c+dOTZ48WS+99JI++uijPPVkZmbqwQcfVGJiotasWVPgvbBpaWkKCAiQhwe3NwMAgKIhRRRk538li/v1A6p0pd/ORQ59aGrv3r0yDEMxMTH5bo+JidGZM2d08uRJSVL79u3twueaNWt0+fJlzZo1S1FRUZKkYcOG6ZVXXrH1efXVVzVt2jT16tVLkhQZGakdO3Zo9uzZ6tevn61fRkaGunXrposXL2rlypUKDAzMt6bTp0/r1Vdf1RNPPFG8kwcAAGUaAbUgl9Iki9sNBlQ36dLZEi/p73JHTi0WiyQpNjY2Tx8/Pz9bOJWksLAwnThxQpJ08uRJ2wNYgwYNsvXJysrKE0D79Omj6tWra8WKFfLz88u3nvT0dHXr1k3169fXhAkTindyAACgTOMSf0F8AiUj58b6GjmST5BDDx8dHS2LxaIdO3bku33Xrl2qUKGCbQqqcuXK5enj6elpt26xWGzBNifnyrnNmTNHmzdvti1//vmn1q9fb/e5rl27auvWrXnac507d05dunSRv7+/vv766zzHBQAAKAwCakFietzY6Kl0pV9MD4cePjg4WB07dtSMGTN08eJFu20pKSmaP3++evfubRtBLawqVaqoWrVqOnDggKKjo+2WyMhIu75PPfWUXnvtNfXs2VOrVq2y25aenq5OnTrJy8tL3333nXx8fIpUDwAAQC4u8RekSgOp+m1Xppi6VlC1uEvVY6Uq9R1ewvTp09WyZUt17txZ8fHxioyM1Pbt2zV27FhVq1ZNkyZNKtb+J06cqBEjRiggIEBxcXGyWq1KTEzUmTNnNHr0aLu+w4cPV3Z2trp3764ffvhBrVu31rlz59SpUydduHBB8+bNU3p6utLT0yVJlSpVkru7e7HqAwA4ztLtKZrx8z7tOZ6hOlX8NaRdtDo3CHV1WUC+CKjX8o9ZV+Y5LWiqKYu75B0g3TuzRA5fu3ZtJSYmauLEierdu7dOnz6t0NBQ3XvvvZowYYIqVqxYrP0PHDhQfn5+mjp1qp599lmVK1dOjRo10qhRo/LtP2rUKOXk5Khr165asmSJMjMz9dtvv0m6ckvC3yUlJalmzZrFqg8A4BhLt6foiU822Na3HEnTk/M2aNYjzQmpMCWLkXtTYimWnp6uwMBA2xRHf3fp0iUlJSUpMjKyaJefT++/Ms9p8m9XAqnF7co9p0a2FH77lXAaHHX9/cCm2L8TAECh3DN9jbYcyfuWv1vCg/Tt0FYuqAhl0bXy2tUYQb2e4ChpwLIr85zuXHTlaX2foCv3nJbAZX0AABxtz/GMfNv3Hj/n5EqAG2OKgFqzZk0dOnQoT/uQIUP07rvvuqCifFRp4NB5TgEAcJY6VfzzHUGtXaW8C6oBrs8UT/H/8ccfOnbsmG358ccfJUkPPPCAiysDAKD0G9IuWldP+mKxSEPbcosazMkUAbVSpUoKDQ21LYsWLVJUVJTatGnj6tIAACj1OjcI1axHmuuW8CD5ebnrlvAgzX6kuTrxgBRMyhSX+P8uMzNT8+bN0+jRowuc49NqtcpqtdrWc6c2AgAA+evcIJQn9lFqmGIE9e+++eYbnT17Vv379y+wz5QpUxQYGGhbwsPDnVcgAAAASpTpAuoHH3yguLg4Va1atcA+48aNU1pamm1JTk52YoUAAAAoSaYKqIcOHdLy5cs1cODAa/bz9vZWQECA3XIze+655xQXFydJatCggWbMmOHiigAAAEqOqe5BnTt3ripXrqxu3bq5uhRTGTt2rO2e28WLFysoKMi1BQEAAJQg0wTUnJwczZ07V/369ZOHh2nKstlzZo9WHFqh9Mx0BXgF6O6Iu1WnQh2nHDskJMT2c0REhFOOKUkHDx5UZGSkNm3apCZNmjjtuAAAoGwzzSX+5cuX6/Dhw3r88cddXYqdw+mH9ejiR3Xfd/dp9tbZ+mz3Z5q9dbbu++4+Pbr4UR1OP1yix09OTtaAAQNUtWpVeXl5KSIiQiNHjtTp06dL9Lg36oknnlBUVJR8fX1VqVIl3XPPPdq1a5erywIAAKWYaQJqp06dZBiG6tRxzqjkjTicflh9vu+jbae2SZKyjWxl5WQp28iWJG07tU19vu9TYiH1wIEDio2N1Z49e7RgwQLt27dPs2bN0ooVK9SiRQulpqbm+7nMzMwSqSc/zZs319y5c7Vz504tXbpUhmGoU6dOys7OdloNAADg5mKagGpG49eM1/nL522B9GrZRrbOXz6vF399sUSOP3ToUHl5eWnZsmVq06aNatSoobi4OC1fvlx//fWXxo8fL+nKq2Lj4+PVv39/BQYGatCgQUpISFBQUJCWLl2qmJgY+fv7q0uXLjp27JjdMebOnauYmBj5+PioXr1613wAKycnR4MGDVKdOnVsr6YdPHiw7rrrLtWsWVPNmjVTfHy8kpOTdfDgwRL5OwEAADc/AmoB9pzZo80nNxcYTnNlG9nadGKT9pzZ49Djp6amaunSpRoyZIh8fX3ttoWGhqpv375auHChDMOQJE2dOlUNGzbUhg0b9NJLL0mSLly4oDfeeEOffPKJVq9ercOHD2vMmDG2/cyZM0fjx4/XpEmTtHPnTk2ePFkvvfSSPvroozz1ZGZm6sEHH1RiYqLWrFmT772w58+f19y5cxUZGcnctAAAoMgIqAVYcWiF3C3uN9TX3eKuFYdXOPT4e/fulWEYiomJyXd7TEyMzpw5o5MnT0qS2rdvrzFjxig6OlrR0dGSpMuXL2vWrFmKjY1Vs2bNNGzYMK1Y8b86X331VU2bNk29evVSZGSkevXqpaefflqzZ8+2O1ZGRoa6deumlJQUrVy5UpUrV7bbPmPGDPn7+8vf319LlizRjz/+KC8vL0f+dQAAgDKEgFqA9Mz0Al+1ejWLxaJ0q3Nft5o7cppbY2xsbJ4+fn5+ioqKsq2HhYXpxIkTkqSTJ0/aHsDKDZf+/v6Kj4/X/v377fbTp08fZWRkaNmyZQoMDMxznL59+2rTpk1atWqVateurQcffFCXLl1y2LkCAICyxXzzOZlEgFeALQRej2EYCvB27MsCoqOjZbFYtGPHDt177715tu/atUsVKlSwTUFVrly5PH08PT3t1i0Wi+2ccnJyJF25zH/77bfb9XN3tx857tq1q+bNm6f169erffv2eY6T+8rZ2rVr64477lCFChX09ddfq0+fPjd+wgAAAP8fI6gFuDvi7uvef5or28hWhxodHHr84OBgdezYUTNmzNDFixfttqWkpGj+/Pnq3bv3DY/yXq1KlSqqVq2aDhw4YLstIHeJjIy06/vUU0/ptddeU8+ePbVq1arr7tswDNuLBQAAAAqLEdQC1KlQR00qNdG2U9uuGVTdLe5qXKmxaleo7fAapk+frpYtW6pz586Kj49XZGSktm/frrFjx6patWqaNGlSsfY/ceJEjRgxQgEBAYqLi5PValViYqLOnDmj0aNH2/UdPny4srOz1b17d/3www9q3bq1Dhw4oIULF6pTp06qVKmS/vrrL/3rX/+Sr6+vunbtWqzaAABA2cUI6jVMaj1J5TzLFfiwlLvFXeU8yym+VXyJHL927dpKTExUVFSUevfuraioKA0ePFjt2rXTunXrVLFixWLtf+DAgXr//feVkJCgRo0aqU2bNkpISMgzgppr1KhRevnll9W1a1etXbtWPj4++uWXX9S1a1dFR0frwQcfVLly5bR27do8D1IBAADcKItxozdamlh6eroCAwOVlpamgAD7e0EvXbqkpKQkRUZGysfHp9D7Ppx+WC/++qI2ndgkd4u77T7ObCNbTSs3VXyreNUIqOGoUykTivs7AQAApc+18trVuMR/HTUCaujjuI+158werTi8QunWdAV4B6hDjQ4lclkfAACgrCOg3qA6FeqoTgXzvIYVAADgZsU9qAAAADAVAioAAABMpcwE1JvgWbCbBr8LAABwLTd9QM19m9KFCxdcXAly5f4urn7TFQAAgFQGHpJyd3dXUFCQ7R30fn5+RX77EorHMAxduHBBJ06cUFBQUJ5XqgIAAEhlIKBKUmhoqCTZQipcKygoyPY7AQAAuFqZCKgWi0VhYWGqXLmyLl++7OpyyjRPT09GTgEAwDWViYCay93dnXAEAABgcjf9Q1IAAAAoXQioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEzFNAH1r7/+0iOPPKLg4GD5+fmpSZMm2rBhg6vLAgAAgJN5uLoASTpz5oxatWqldu3a6YcfflDlypW1f/9+BQUFubo0AAAAOJkpAuq//vUvhYeHa+7cuba2mjVrFtjfarXKarXa1tPT00uyPAAAADiRKS7xf/fdd4qNjdUDDzygypUrq2nTppozZ06B/adMmaLAwEDbEh4e7sRqAQAAUJIshmEYri7Cx8dHkjR69Gg98MAD+v333zVq1CjNnj1b//znP/P0z28ENTw8XGlpaQoICHBa3QAAALgx6enpCgwMvKG8ZoqA6uXlpdjYWK1du9bWNmLECP3xxx9at27ddT9fmBMGAACA8xUmr5niEn9YWJjq169v1xYTE6PDhw+7qCIAAAC4iikCaqtWrbR79267tj179igiIsJFFQEAAMBVTBFQn376aa1fv16TJ0/Wvn379Omnn+q9997T0KFDXV0aAAAAnMwUAfXWW2/V119/rQULFqhhw4Z69dVX9dZbb6lv376uLg0AAABOZoqHpIqLh6QAAADMrdQ9JAUAAADkIqACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABT8XB1AQAAQNIlq3Q5y9VVAP/j6SH5eLvk0ARUAABc7ZJVOX/8KbecUv/2cdxEctwscru1oUtCKgEVAABXu5wltxxDP3r6K9XN3dXVAKqYk62OlzOujOoTUAEAKLtS3dx1yo3/NAM8JAUAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTKdTrKrZu3aqGDRvKzc1NW7duvWbfxo0bF6swAAAAlE2FCqhNmjRRSkqKKleurCZNmshiscgwDNv23HWLxaLs7GyHFwsAAICbX6ECalJSkipVqmT7GQAAAHC0QgXUiIiIfH9G8S3dnqIZP+/TnuMZqlPFX0PaRatzg1BXl4UCrDi8Qu9vfV/70/YrKjBKAxsP1N017nZ1WShtdi6SfpkmndwlVaon3fmMFNPd1VUBgMsV+SGpjz76SN9//71t/dlnn1VQUJBatmypQ4cOOaS4smLp9hQ98ckGbTmSpouXs7XlSJqenLdBS7enuLo05GPF4RUa9fMo/Xn6T13Muqg/T/+pp39+WisOr3B1aShNdi6SFvaVjm6ULl+48ufCR660A0AZV+SAOnnyZPn6+kqS1q1bp+nTp+v1119XSEiInn76aYcVWBbM+HlfnjbDkGas3O+CanA97299P0+bIUMfbPvABdWg1PplWj6NhrTm304vBQDMplCX+P8uOTlZ0dHRkqRvvvlG999/vwYPHqxWrVqpbdu2jqqvTNhzPCPf9r3Hzzm5EtyI/Wn5/8Nh39m8/9AACnRyV/7tJwpoB4AypMgjqP7+/jp9+rQkadmyZerQoYMkycfHRxcvXnRMdWVEnSr++bbXrlLeyZXgRkQFRuXbHh0U7eRKUKpVqpd/e+UC2gGgDClyQO3YsaMGDhyogQMHas+ePerWrZskafv27apZs6aj6isThrSLlsVi32axSEPb5h+E4FoDGw+URfa/MIssGthooIsqQql05zPSVf87kiz/vx0AyrYiB9R3331XLVq00MmTJ/Xll18qODhYkrRhwwb16dPHYQWWBZ0bhGrWI811S3iQ/LzcdUt4kGY/0lydeIrflO6ucbfebPemGoU0kq+HrxqFNNJb7d5S+xrtXV0aSpOY7lLveVK15pJnuSt/PjRfqtfN1ZUBgMtZjL/PtF9KpaenKzAwUGlpaQoICHB1OQAAFM6589LGnVroHahTbkV+PARwmJCcLPW2pknNYqTy5Ryyz8LktWJ9C3755RfNnj1bBw4c0Oeff65q1arpk08+UWRkpFq3bn3D+5k4caJefvllu7YqVaooJcXF0yxdskqXs1xbA/B3nh6Sj7erqwAAoEQVOaB++eWXevTRR9W3b19t3LhRVqtVknTu3DlNnjxZixcvLtT+GjRooOXLl9vW3d3di1qaY1yyKuePP+WWU+oHmHETyXGzyO3WhoRUAMBNrcgBNT4+XrNmzdI///lPffbZZ7b2li1b6pVXXil8IR4eCg010T2Xl7PklmPoR09/pbq5OCwDkirmZKvj5Ywro/oEVADATazIAXX37t2666678rQHBATo7Nmzhd7f3r17VbVqVXl7e+v222/X5MmTVatWrXz7Wq1W24itdOWehpKS6ubO/UAAAABOVOSn+MPCwrRvX96JydesWVNgsCzI7bffro8//lhLly7VnDlzlJKSopYtW9rmWb3alClTFBgYaFvCw8OLdA4AAAAwnyIH1CeeeEIjR47Ub7/9JovFoqNHj2r+/PkaM2aMhgwZUqh9xcXF6b777lOjRo3UoUMHff/995Kkjz76KN/+48aNU1pamm1JTk4u6mkAAADAZIp87frZZ59VWlqa2rVrp0uXLumuu+6St7e3xowZo2HDhhWrqHLlyqlRo0bau3dvvtu9vb3l7c09eAAAADejYt1cOWnSJI0fP147duxQTk6O6tevL3///F/bWRhWq1U7d+7UnXfeWex9AQAAoHQp8iX+hIQEXbx4UX5+foqNjdVtt91W5HA6ZswYrVq1SklJSfrtt990//33Kz09Xf369StqeQAAACilihxQx40bpypVqmjAgAFau3ZtsYo4cuSI+vTpo7p166pXr17y8vLS+vXrFRERUaz9AgAAoPQp8iX+I0eO6Pvvv1dCQoLatWunyMhIPfbYY+rXr1+h5zP9+zyquHlEpvyk5vs/UMWM/Ur1j9KGqAFKCuV99QAA4NqKPILq7u6unj176quvvlJycrIGDx6s+fPnq0aNGurZs6e+/fZb5eTkOLJWlCKRKT+p68bRqpK2XZ7Zl1QlbbviNj6jyJSfXF0aAAAwuSIH1L+rXLmyWrVqpRYtWsjNzU3btm1T//79FRUVpZUrVzriEChlmu//IE+bRYaa7//QBdUAAIDSpFgB9fjx43rjjTfUoEEDtW3bVunp6Vq0aJGSkpJ09OhR9erViwedyqiKGfsL1Q4AAJCryAG1R48eCg8PV0JCggYNGqS//vpLCxYsUIcOHSRJvr6+euaZZ5hEv4xK9Y8qVDsAAECuIj8kVblyZa1atUotWrQosE9YWJiSkpKKegiUYhuiBihu4zOyyLC1GbJoQ/QAF1YFAABKgyIH1A8+yHuP4dUsFgtTRZVRSaHt9UOzaWq+/8P/PcUfPUBJVdq5ujQAAGByhQqo77zzjgYPHiwfHx+988471+w7YsSIYhWG0i8ptD3TSgEAgEIrVEB988031bdvX/n4+OjNN98ssJ/FYiGgAgAAoEgKFVD/fj8p95YCAACgJBT5Kf5Vq1Y5sg4AAABAUjECaseOHVWjRg09//zz2rZtmyNrAgAAQBlW5IB69OhRPfvss/rll190yy23qHHjxnr99dd15MgRR9YHAACAMqbIATUkJETDhg3Tr7/+qv3796t37976+OOPVbNmTbVvz5PbAAAAKJpiveo0V2RkpJ5//nm99tpratSoEfenAgAAoMiKHVB//fVXDRkyRGFhYXr44YfVoEEDLVq0yBG1AQAAoAwq8pukXnjhBS1YsEBHjx5Vhw4d9NZbb+nee++Vn5+fI+sDAABAGVPkgLpy5UqNGTNGvXv3VkhIiCNrAgAAQBlW5IC6du1aR9YBAAAASCrmPaiffPKJWrVqpapVq+rQoUOSpLfeekvffvutQ4oDAABA2VPkgDpz5kyNHj1aXbt21dmzZ5WdnS1JCgoK0ltvveWo+gAAAFDGFDmg/uc//9GcOXM0fvx4ubu729pjY2N5sxQAAACKrMgBNSkpSU2bNs3T7u3trfPnzxerKAAAAJRdRQ6okZGR2rx5c572H374QfXr1y9OTQAAACjDivwU/9ixYzV06FBdunRJhmHo999/14IFCzRlyhS9//77jqwRAAAAZUiRA+pjjz2mrKwsPfvss7pw4YIefvhhVatWTW+//bYeeughR9YIAACAMqTIAVWSBg0apEGDBunUqVPKyclR5cqVHVUXAAAAyqhizYOaKyQkRDt37tQPP/ygM2fOOGKXAAAAKKMKPYI6depUZWRk6OWXX5YkGYahuLg4LVu2TJJUuXJlrVixQg0aNHBspQAAACgTCj2CumDBArun9L/44gutXr1av/zyi06dOqXY2FhbeAUAAAAKq9ABNSkpSY0bN7atL168WPfdd59atWqlihUr6sUXX9S6descWiQAAADKjkIH1MuXL8vb29u2vm7dOrVs2dK2XrVqVZ06dcox1QEAAKDMKXRAjY6O1urVqyVJhw8f1p49e9SmTRvb9iNHjig4ONhxFQIAAKBMKfRDUk899ZSGDRumX375RevXr1eLFi3s7kn96aef8n0FKgAAAHAjCh1Qn3jiCXl4eGjRokW66667NGHCBLvtR48e1eOPP+6wAgEAAFC2FGke1AEDBujrr7/WzJkzFRoaardtxowZ+sc//lHkgqZMmSKLxaJRo0YVeR8AAAAovRwyUb+j/PHHH3rvvffsZgkAAABA2WKagJqRkaG+fftqzpw5qlChgqvLAQAAgIuYJqAOHTpU3bp1U4cOHa7b12q1Kj093W4BAADAzaHQD0mVhM8++0wbNmxQYmLiDfWfMmUKb6sCAAC4SRV7BHXfvn1aunSpLl68KEkyDKNQn09OTtbIkSM1f/58+fj43NBnxo0bp7S0NNuSnJxc6LoBAABgTkUeQT19+rR69+6tn376SRaLRXv37lWtWrU0cOBABQUFadq0aTe0nw0bNujEiRNq3ry5rS07O1urV6/W9OnTZbVa5e7ubvcZb29vu7dZAQAA4OZR5BHUp59+Wh4eHjp8+LD8/Pxs7b1799aSJUtueD933323tm3bps2bN9uW2NhY9e3bV5s3b84TTgEAAHBzK/II6rJly7R06VJVr17drr127do6dOjQDe+nfPnyatiwoV1buXLlFBwcnKcdAAAAN78ij6CeP3/ebuQ016lTp7j8DgAAgCIrckC966679PHHH9vWLRaLcnJyNHXqVLVr165YRa1cuVJvvfVWsfYBAACA0qnIl/inTp2qtm3bKjExUZmZmXr22We1fft2paam6tdff3VkjQAAAChDijyCWr9+fW3dulW33XabOnbsqPPnz6tXr17atGmToqKiHFkjAAAAypBiTdQfGhrKhPkAAABwqEIF1K1bt95w38aNGxe6GAAAAKBQAbVJkyayWCwyDEMWi8XWnvv2qL+3ZWdnO6hEAAAAlCWFugc1KSlJBw4cUFJSkr788ktFRkZqxowZtgn2Z8yYoaioKH355ZclVS8AAABucoUaQY2IiLD9/MADD+idd95R165dbW2NGzdWeHi4XnrpJd17770OKxIAAABlR5Gf4t+2bZsiIyPztEdGRmrHjh3FKgoAAABlV5EDakxMjOLj43Xp0iVbm9VqVXx8vGJiYhxSHAAAAMqeIk8zNWvWLPXo0UPh4eG65ZZbJElbtmyRxWLRokWLHFYgAAAAypYiB9TbbrtNSUlJmjdvnnbt2iXDMNS7d289/PDDKleunCNrBFCaXLJKl7NcXQVgz9ND8vF2dRUAblCxJur38/PT4MGDHVULgNLuklU5f/wptxzD1ZUAdnLcLHK7tSEhFSglihVQJWnHjh06fPiwMjMz7dp79uxZ3F0DKG0uZ8ktx9CPnv5KdXN3dTWAJKliTrY6Xs64MrJPQAVKhSIH1AMHDugf//iHtm3bZpu8X/rfZP1M1A+UXalu7jrlVux//wIAyqgiP8U/cuRIRUZG6vjx4/Lz89P27du1evVqxcbGauXKlQ4sEQAAAGVJkYc41q1bp59++kmVKlWSm5ub3Nzc1Lp1a02ZMkUjRozQpk2bHFknAAAAyogij6BmZ2fL399fkhQSEqKjR49KuvK2qd27dzumOgAAAJQ5RR5BbdiwobZu3apatWrp9ttv1+uvvy4vLy+99957qlWrliNrBFDG7T2arvW7T+l0ulXBAd66o26IalcNcHVZAIASUuQR1BdffFE5OTmSpPj4eB06dEh33nmnFi9erHfeecdhBQIo2/YeTdc3648o5cwlXc42lHLmkr5Zf0R7j6a7ujQAQAkp1Ajq1q1b1bBhQ7m5ualz58629lq1amnHjh1KTU1VhQoVbE/yA0Bxrd99Kt/233afYhQVAG5ShRpBbdq0qU6duvIfi1q1aun06dN22ytWrEg4BeBQp9Ot+bafKqAdAFD6FSqgBgUFKSkpSZJ08OBB2yV+ACgpwQH5T6weUkA7AKD0K9Ql/vvuu09t2rRRWFiYLBaLYmNj5e6e/9tiDhw44JACAZRtd9QN0Tfrj+Rpv71uJRdUAwBwhkIF1Pfee0+9evXSvn37NGLECA0aNEjly5cvqdoAQLWrBujeO6rrt92ndCrdqpAAb91et5JqV+X/ewDgZlXoaaa6dOkiSdqwYYNGjhxJQAVQ4mpXDeCBKAAoQ4o8D+rcuXMdWQcAAAAg6QYDat++ffXmm2+qcuXK6tWr1w3t+KuvvipWYQAAACibbiigBgQEyM3tygP/gYGBJVoQAAAAyrYbCqgzZ860/cylfQAAAJSkIr/qFAAAACgJBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqpgioM2fOVOPGjRUQEKCAgAC1aNFCP/zwg6vLAgAAgAuYIqBWr15dr732mhITE5WYmKj27dvrnnvu0fbt211dGgAAAJysyK86daQePXrYrU+aNEkzZ87U+vXr1aBBAxdVBQAAAFcwRUD9u+zsbH3++ec6f/68WrRokW8fq9Uqq9VqW09PT3dWeQAAAChhprjEL0nbtm2Tv7+/vL299eSTT+rrr79W/fr18+07ZcoUBQYG2pbw8HAnVwsAAICSYpqAWrduXW3evFnr16/XU089pX79+mnHjh359h03bpzS0tJsS3JyspOrBQAAQEkxzSV+Ly8vRUdHS5JiY2P1xx9/6O2339bs2bPz9PX29pa3t7ezSwQAAIATmGYE9WqGYdjdZwoAAICywRQjqC+88ILi4uIUHh6uc+fO6bPPPtPKlSu1ZMkSV5cGAAAAJzNFQD1+/LgeffRRHTt2TIGBgWrcuLGWLFmijh07uro0AAAAOJkpAuoHH3zg6hIAAABgEqa9BxUAAABlEwEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYioerCwAAlG1J59Zr46nPdcZ6WBW8a6hZyAOKLH+Hq8sC4EKMoAIAXCbp3HotPTJZJy/tVZZh1clLe7X0yBQlnVvv6tIAuBABFQDgMhtPfZ5Pq6FNp75wei0AzIOACgBwmTPWw/m2pxbQDqBsIKACAFymgneNfNsrFtAOoGwgoAIAXKZZyAOSLFe1WtQ05H5XlAPAJAioAACXiSx/hzpXH6fKPnXkYfFRZZ866lx9HE/xA2Uc00wBAFwqsvwdBFIAdkwxgjplyhTdeuutKl++vCpXrqx7771Xu3fvdnVZAAAAcAFTBNRVq1Zp6NChWr9+vX788UdlZWWpU6dOOn/+vKtLAwAAgJOZ4hL/kiVL7Nbnzp2rypUra8OGDbrrrrtcVBUAAABcwRQB9WppaWmSpIoVK+a73Wq1ymq12tbT09OdUhcAAABKniku8f+dYRgaPXq0WrdurYYNG+bbZ8qUKQoMDLQt4eHhTq4SAAAAJcV0AXXYsGHaunWrFixYUGCfcePGKS0tzbYkJyc7sUIAAACUJFNd4h8+fLi+++47rV69WtWrVy+wn7e3t7y9vZ1YGQAAAJzFFAHVMAwNHz5cX3/9tVauXKnIyEhXlwQAAAAXMUVAHTp0qD799FN9++23Kl++vFJSUiRJgYGB8vX1dXF1AAAAcCZT3IM6c+ZMpaWlqW3btgoLC7MtCxcudHVpAAAAcDJTjKAahuHqEgAAAGASphhBBQAAAHIRUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKl4uLoAAAAA3Likc+u18dTnOmM9rAreNdQs5AFFlr/D1WU5FCOoAAAApUTSufVaemSyTl7aqyzDqpOX9mrpkSlKOrfe1aU5FAEVAACglNh46vN8Wg1tOvWF02spSQRUAACAUuKM9XC+7akFtJdWBFQAAIBSooJ3jXzbKxbQXloRUAEAAEqJZiEPSLJc1WpR05D7XVFOiSGgAgAAlBKR5e9Q5+rjVNmnjjwsPqrsU0edq4+76Z7iZ5opAACAUiSy/B03XSC9GiOoAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBUCKgAAAEyFgAoAAABTIaACAADAVAioAAAAMBVTBNTVq1erR48eqlq1qiwWi7755htXlwQAAAAXMUVAPX/+vG655RZNnz7d1aUAAADAxTxcXYAkxcXFKS4uztVlAAAAwARMEVALy2q1ymq12tbT09NdWA0AAAAcyRSX+AtrypQpCgwMtC3h4eGuLgkAAAAOUioD6rhx45SWlmZbkpOTXV0SAAAAHKRUXuL39vaWt7e3q8sAAABACSiVI6gAAAC4eZliBDUjI0P79u2zrSclJWnz5s2qWLGiatSo4cLKAAAA4GymCKiJiYlq166dbX306NGSpH79+ikhIcFFVQEAAMAVTBFQ27ZtK8MwXF0GAAAATIB7UAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYCgEVAAAApkJABQAAgKkQUAEAAGAqBFQAAACYimkC6owZMxQZGSkfHx81b95cv/zyi6tLAgAAgAuYIqAuXLhQo0aN0vjx47Vp0ybdeeediouL0+HDh11dGgAAAJzMFAH13//+twYMGKCBAwcqJiZGb731lsLDwzVz5kxXlwYAAAAn83B1AZmZmdqwYYOef/55u/ZOnTpp7dq1+X7GarXKarXa1tPS0iRJ6enpjivs3HnpfIa8MyU/N3fH7RcoIu+cbKVfzpDS0yUj29Xl5I/vDUyI7w5QeCXxvcnNaYZhXLevywPqqVOnlJ2drSpVqti1V6lSRSkpKfl+ZsqUKXr55ZfztIeHh5dIjQAAAHCMc+fOKTAw8Jp9XB5Qc1ksFrt1wzDytOUaN26cRo8ebVvPyclRamqqgoODC/wMXCM9PV3h4eFKTk5WQECAq8sBSg2+O0DR8N0xL8MwdO7cOVWtWvW6fV0eUENCQuTu7p5ntPTEiRN5RlVzeXt7y9vb264tKCiopEqEAwQEBPB/FEAR8N0Biobvjjldb+Q0l8sfkvLy8lLz5s31448/2rX/+OOPatmypYuqAgAAgKu4fARVkkaPHq1HH31UsbGxatGihd577z0dPnxYTz75pKtLAwAAgJOZIqD27t1bp0+f1iuvvKJjx46pYcOGWrx4sSIiIlxdGorJ29tbEyZMyHNLBoBr47sDFA3fnZuDxbiRZ/0BAAAAJ3H5PagAAADA3xFQAQAAYCoEVAAAAJgKARUAAACmQkDFdaWkpGjkyJGKjo6Wj4+PqlSpotatW2vWrFm6cOGCw4+3du1aubu7q0uXLg7fN+BMzvru9O/fXxaLxbYEBwerS5cu2rp1q8OOATiTM/+7k5KSouHDh6tWrVry9vZWeHi4evTooRUrVjj0OCgcU0wzBfM6cOCAWrVqpaCgIE2ePFmNGjVSVlaW9uzZow8//FBVq1ZVz549HXrMDz/8UMOHD9f777+vw4cPq0aNGg7dP+AMzv7udOnSRXPnzpV05T+4L774orp3767Dhw877BiAMzjzu3Pw4EHbsV5//XU1btxYly9f1tKlSzV06FDt2rXLIcdBERjANXTu3NmoXr26kZGRke/2nJwcwzAM4+zZs8agQYOMSpUqGeXLlzfatWtnbN682dZvwoQJxi233GJ8/PHHRkREhBEQEGD07t3bSE9Pt9tfRkaGUb58eWPXrl1G7969jZdffrnkTg4oQc787vTr18+455577Pa/evVqQ5Jx4sQJx58cUIKc+d2Ji4szqlWrlu+xzpw549gTQ6FwiR8FOn36tJYtW6ahQ4eqXLly+faxWCwyDEPdunVTSkqKFi9erA0bNqhZs2a6++67lZqaauu7f/9+ffPNN1q0aJEWLVqkVatW6bXXXrPb38KFC1W3bl3VrVtXjzzyiObOnSuDqXpRyrjiu/N3GRkZmj9/vqKjoxUcHOzw8wNKijO/O6mpqVqyZEmBxwoKCiqRc8QNcm0+hpmtX7/ekGR89dVXdu3BwcFGuXLljHLlyhnPPvussWLFCiMgIMC4dOmSXb+oqChj9uzZhmFc+Zesn5+f3b9cx44da9x+++12n2nZsqXx1ltvGYZhGJcvXzZCQkKMH3/8sSRODygxzv7u9OvXz3B3d7ftW5IRFhZmbNiwoQTPEnA8Z353fvvtt3yPBXPgHlRcl8VisVv//ffflZOTo759+8pqtWrDhg3KyMjIM1Jz8eJF7d+/37Zes2ZNlS9f3rYeFhamEydO2NZ3796t33//XV999ZUkycPDQ71799aHH36oDh06lMSpASXKWd8dSWrXrp1mzpwp6crI0IwZMxQXF6fff/+d10aj1HHGd8f4/1fnrj4WzIGAigJFR0fLYrHkuUm8Vq1akiRfX19JUk5OjsLCwrRy5co8+/j7JRJPT0+7bRaLRTk5Obb1Dz74QFlZWapWrZqtzTAMeXp66syZM6pQoUJxTwlwCmd/dySpXLlyio6Otq03b95cgYGBmjNnjuLj44tzOoDTOPO7U7t2bVksFu3cuVP33nuv404CDsE9qChQcHCwOnbsqOnTp+v8+fMF9mvWrJlSUlLk4eGh6OhouyUkJOSGjpWVlaWPP/5Y06ZN0+bNm23Lli1bFBERofnz5zvqtIAS58zvTkEsFovc3Nx08eLFYu0HcCZnfncqVqyozp0769133833WGfPni3qacABCKi4phkzZigrK0uxsbFauHChdu7cqd27d2vevHnatWuX3N3d1aFDB7Vo0UL33nuvli5dqoMHD2rt2rV68cUXlZiYeEPHWbRokc6cOaMBAwaoYcOGdsv999+vDz74oITPFHAsZ313clmtVqWkpCglJUU7d+7U8OHDlZGRoR49epTQGQIlw5nfnRkzZig7O1u33XabvvzyS+3du1c7d+7UO++8oxYtWpTgWeJ6uMSPa4qKitKmTZs0efJkjRs3TkeOHJG3t7fq16+vMWPGaMiQIbJYLFq8eLHGjx+vxx9/XCdPnlRoaKjuuusuValS5YaO88EHH6hDhw4KDAzMs+2+++7T5MmTtXHjRjVr1szRpwiUCGd9d3ItWbJEYWFhkqTy5curXr16+vzzz9W2bdsSODug5DjzuxMZGamNGzdq0qRJeuaZZ3Ts2DFVqlRJzZs3t93TDdewGAZz+AAAAMA8uMQPAAAAUyGgAgAAwFQIqAAAADAVAioAAABMhYAKAAAAUyGgAgAAwFQIqAAAADAVAioAAABMhYAKAAAAUyGgAoATpaSkaOTIkYqOjpaPj4+qVKmi1q1ba9asWbpw4YKrywMAU/BwdQEAUFYcOHBArVq1UlBQkCZPnqxGjRopKytLe/bs0YcffqiqVauqZ8+eri4TAFyOEVQAcJIhQ4bIw8NDiYmJevDBBxUTE6NGjRrpvvvu0/fff68ePXpIktLS0jR48GBVrlxZAQEBat++vbZs2WLbz8SJE9WkSRN98sknqlmzpgIDA/XQQw/p3Llzrjo1AHAoAioAOMHp06e1bNkyDR06VOXKlcu3j8VikWEY6tatm1JSUrR48WJt2LBBzZo10913363U1FRb3/379+ubb77RokWLtGjRIq1atUqvvfaas04HAEoUARUAnGDfvn0yDEN169a1aw8JCZG/v7/8/f313HPP6eeff9a2bdv0+eefKzY2VrVr19Ybb7yhoKAgffHFF7bP5eTkKCEhQQ0bNtSdd96pRx99VCtWrHD2aQFAieAeVABwIovFYrf++++/KycnR3379pXVatWGDRuUkZGh4OBgu34XL17U/v37bes1a9ZU+fLlbethYWE6ceJEyRYPAE5CQAUAJ4iOjpbFYtGuXbvs2mvVqiVJ8vX1lXRlZDQsLEwrV67Ms4+goCDbz56ennbbLBaLcnJyHFs0ALgIARUAnCA4OFgdO3bU9OnTNXz48ALvQ23WrJlSUlLk4eGhmjVrOrdIADAJ7kEFACeZMWOGsrKyFBsbq4ULF2rnzp3avXu35s2bp127dsnd3V0dOnRQixYtdO+992rp0qU6ePCg1q5dqxdffFGJiYmuPgUAcApGUAHASaKiorRp0yZNnjxZ48aN05EjR+Tt7a369etrzJgxGjJkiCwWixYvXqzx48fr8ccf18mTJxUaGqq77rpLVapUcfUpAIBTWAzDMFxdBAAAAJCLS/wAAAAwFQIqAAAATIWACgAAAFMhoAIAAMBUCKgAAAAwFQIqAAAATIWACgAAAFMhoAIAAMBUCKgAAAAwFQIqAAAATIWACgAAAFP5f5l+ROwrqSoXAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 800x500 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn as sns\n",
    "\n",
    "data= {\n",
    "    'Gen' :['GenA','GenB','GenC'],\n",
    "    'rnek1':[5.2,3.4,7.1],\n",
    "    'rnek2':[4.8,5.2,8.2],\n",
    "    'rnek3':[5.2,2.2,1.5],\n",
    "}\n",
    "\n",
    "df = pd.DataFrame(data)\n",
    "\n",
    "df_long = df.melt(id_vars='Gen', var_name = 'rnek',value_name='fade')\n",
    "\n",
    "plt.figure(figsize=(8,5))\n",
    "\n",
    "sns.barplot(data=df_long,x='Gen', y='fade',ci =None,color='skyblue',edgecolor='pink')\n",
    "\n",
    "sns.stripplot(data=df_long, x='Gen', y='fade',hue='rnek',dodge=True,jitter=True,marker='o',edgecolor='black')\n",
    "\n",
    "plt.title('Gen fadesi Daglm')\n",
    "plt.ylabel('fade Seviyesi')\n",
    "plt.legend(title='rnek')\n",
    "plt.show()\n",
    "\n",
    "#melt() = tabloyu uzun forma eviriyoruz(gen baznda rnekler tm satrda olsun diye)\n",
    "#barplot ile genlerin ortlama ifadesini alyoz\n",
    "#stripplot ile her rnein tek tek deerini noktalarla ekledik\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d640048f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
